package io.github.up2jakarta.cii.edi;

import io.github.up2jakarta.cii.Documented;
import io.github.up2jakarta.cii.core.Agency;
import io.github.up2jakarta.cii.edi.adapters.DocumentCodeAdapter;
import io.github.up2jakarta.lov.CodeList;
import io.github.up2jakarta.lov.Schema;
import jakarta.xml.bind.annotation.adapters.XmlJavaTypeAdapter;

import javax.annotation.processing.Generated;

/**
 * Based on UN/CEFACT 1001 : Document name code.
 * {@see https://service.unece.org/trade/untdid/d22b/tred/tred1001.htm}
 */
@Generated(value = "CII", comments = "by A.ABBESSI")
@Documented(value = "Document Name Code", agency = Agency.UN_ECE, version = "D22A")
@Schema(agency = "UN/CEFACT", version = "3.17", date = "2008-08-23")
@XmlJavaTypeAdapter(DocumentCodeAdapter.class)
public enum DocumentCodeType implements CodeList<DocumentCodeType> {

    /**
     * Certificate providing the values of an analysis.
     */
    V_1("1", "Certificate of analysis"),

    /**
     * Certificate certifying the conformity to predefined definitions.
     */
    V_2("2", "Certificate of conformity"),

    /**
     * Certificate certifying the quality of goods, services etc.
     */
    V_3("3", "Certificate of quality"),

    /**
     * Report providing the results of a test session.
     */
    V_4("4", "Test report"),

    /**
     * Report specifying the performance values of products.
     */
    V_5("5", "Product performance report"),

    /**
     * Report providing specification values of products.
     */
    V_6("6", "Product specification report"),

    /**
     * Reports on events during production process.
     */
    V_7("7", "Process data report"),

    /**
     * Document/message describes the test report of the first sample.
     */
    V_8("8", "First sample test report"),

    /**
     * A document/message to enable the transmission of information regarding pricing and catalogue
     * details for goods and services offered by a seller to a buyer.
     */
    V_9("9", "Price/sales catalogue"),

    /**
     * Document/message providing basic data concerning a party.
     */
    V_10("10", "Party information"),

    /**
     * A pre-approved document relating to federal label approval requirements.
     */
    V_11("11", "Federal label approval"),

    /**
     * Certificate certifying a specific quality of agricultural products.
     */
    V_12("12", "Mill certificate"),

    /**
     * Document/message which evidences the transport of goods by post (e.g. mail, parcel,
     * etc.).
     */
    V_13("13", "Post receipt"),

    /**
     * Certificate certifying the weight of goods.
     */
    V_14("14", "Weight certificate"),

    /**
     * Document/message specifying the weight of goods.
     */
    V_15("15", "Weight list"),

    /**
     * Document by means of which the documentary credit applicant specifies the conditions
     * for the certificate and by whom the certificate is to be issued.
     */
    V_16("16", "Certificate"),

    /**
     * Document identifying goods in which the issuing authority expressly certifies that the
     * goods originate in a specific country or part of, or group of countries. It also states
     * the price and/or cost of the goods with the purpose of determining the customs origin.
     */
    V_17("17", "Combined certificate of value and origin"),

    /**
     * Specific form of transit declaration issued by the exporter (movement certificate).
     */
    V_18("18", "Movement certificate A.TR.1"),

    /**
     * Certificate certifying the quantity of goods, services etc.
     */
    V_19("19", "Certificate of quantity"),

    /**
     * Usage of QALITY-message.
     */
    V_20("20", "Quality data message"),

    /**
     * Request information based on defined criteria.
     */
    V_21("21", "Query"),

    /**
     * Document/message returned as an answer to a question.
     */
    V_22("22", "Response to query"),

    /**
     * Information regarding the status of a related message.
     */
    V_23("23", "Status information"),

    /**
     * Message/document identifying containers that have been unloaded and then reloaded onto
     * the same means of transport.
     */
    V_24("24", "Restow"),

    /**
     * Message/document itemising containers to be discharged from vessel.
     */
    V_25("25", "Container discharge list"),

    /**
     * Document/message providing contributions advice used for corporate superannuation schemes.
     */
    V_26("26", "Corporate superannuation contributions advice"),

    /**
     * Document/message providing contributions advice used for superannuation schemes which
     * are industry wide.
     */
    V_27("27", "Industry superannuation contributions advice"),

    /**
     * Member maintenance message used for corporate superannuation schemes.
     */
    V_28("28", "Corporate superannuation member maintenance message"),

    /**
     * Member maintenance message used for industry wide superannuation schemes.
     */
    V_29("29", "Industry superannuation member maintenance message"),

    /**
     * Payroll deductions advice used in the life insurance industry.
     */
    V_30("30", "Life insurance payroll deductions advice"),

    /**
     * A Message/document requesting to move cargo from one Customs control point to another.
     */
    V_31("31", "Underbond request"),

    /**
     * A message/document issuing Customs approval to move cargo from one Customs control point
     * to another.
     */
    V_32("32", "Underbond approval"),

    /**
     * Document / message issued by the authority in the exporting country evidencing the sealing
     * of export meat lockers.
     */
    V_33("33", "Certificate of sealing of export meat lockers"),

    /**
     * Message identifying the status of cargo.
     */
    V_34("34", "Cargo status"),

    /**
     * A message specifying information relating to held inventories.
     */
    V_35("35", "Inventory report"),

    /**
     * Official document to identify a person.
     */
    V_36("36", "Identity card"),

    /**
     * Document/message in which the competent national authorities provide a declarant with
     * an acceptance or a rejection about a received declaration for European statistical purposes.
     */
    V_37("37", "Response to a trade statistics message"),

    /**
     * Official document proving immunisation against certain diseases.
     */
    V_38("38", "Vaccination certificate"),

    /**
     * An official document giving permission to travel in foreign countries.
     */
    V_39("39", "Passport"),

    /**
     * An official document giving permission to drive a vehicle in a given country.
     */
    V_40("40", "Driving licence (national)"),

    /**
     * An official document giving a native of one country permission to drive a vehicle in
     * certain other countries.
     */
    V_41("41", "Driving licence (international)"),

    /**
     * A document giving free access to a service.
     */
    V_42("42", "Free pass"),

    /**
     * A document giving access to a service for a determined period of time.
     */
    V_43("43", "Season ticket"),

    /**
     * (1125) A message to report the transport status and/or change in the transport status
     * (i.e. event) between agreed parties.
     */
    V_44("44", "Transport status report"),

    /**
     * (1127) A message to request a transport status report (e.g. through the national multimodal
     * status report message IFSTA).
     */
    V_45("45", "Transport status request"),

    /**
     * A banking status document and/or message.
     */
    V_46("46", "Banking status"),

    /**
     * Document/message in which a declarant provides information about extra-Community trade
     * of goods required by the body responsible for the collection of trade statistics. Trade
     * by a country in the European Union with a country outside the European Union.
     */
    V_47("47", "Extra-Community trade statistical declaration"),

    /**
     * Written instructions relating to dangerous goods and defined in the European Agreement
     * of Dangerous Transport by Road known as ADR (Accord europeen relatif au transport international
     * des marchandises Dangereuses par Route).
     */
    V_48("48", "Written instructions in conformance with ADR article number 10385"),

    /**
     * Official certification that damages to the goods to be transported have been discovered.
     */
    V_49("49", "Damage certification"),

    /**
     * A validated priced tender.
     */
    V_50("50", "Validated priced tender"),

    /**
     * A document providing a response to a previously sent price/sales catalogue.
     */
    V_51("51", "Price/sales catalogue response"),

    /**
     * A document providing the result of price negotiations.
     */
    V_52("52", "Price negotiation result"),

    /**
     * Document or message to supply advice on a dangerous or hazardous material to industrial
     * customers so as to enable them to take measures to protect their employees and the environment
     * from any potential harmful effects from these material.
     */
    V_53("53", "Safety and hazard data sheet"),

    /**
     * A statement of an account containing the booked items as in the ledger of the account
     * servicing financial institution.
     */
    V_54("54", "Legal statement of an account"),

    /**
     * A statement from the account servicing financial institution containing items pending
     * to be booked.
     */
    V_55("55", "Listing statement of an account"),

    /**
     * Last statement of a period containing the interest calculation and the final balance
     * of the last entry date.
     */
    V_56("56", "Closing statement of an account"),

    /**
     * Report on the movement of containers or other items of transport equipment to record
     * physical movement activity and establish the beginning of a rental period.
     */
    V_57("57", "Transport equipment on-hire report"),

    /**
     * Report on the movement of containers or other items of transport equipment to record
     * physical movement activity and establish the end of a rental period.
     */
    V_58("58", "Transport equipment off-hire report"),

    /**
     * No shortage, surplus or damaged outturn resulting from container vessel unpacking.
     */
    V_59("59", "Treatment - nil outturn"),

    /**
     * Movement type indicator: goods are moved under customs control for warehousing due to
     * being time-up.
     */
    V_60("60", "Treatment - time-up underbond"),

    /**
     * Movement type indicator: goods are to move by sea under customs control to a customs
     * office where formalities will be completed.
     */
    V_61("61", "Treatment - underbond by sea"),

    /**
     * Cargo consists of personal effects.
     */
    V_62("62", "Treatment - personal effect"),

    /**
     * Cargo consists of timber.
     */
    V_63("63", "Treatment - timber"),

    /**
     * Document/message issued either by a factor to indicate his preliminary credit assessment
     * on a buyer, or by a seller to request a factor's preliminary credit assessment on a
     * buyer.
     */
    V_64("64", "Preliminary credit assessment"),

    /**
     * Document/message issued either by a factor to give a credit cover on a buyer, or by
     * a seller to request a factor's credit cover.
     */
    V_65("65", "Credit cover"),

    /**
     * Document/message issued by a factor to indicate the money movements of a seller's or
     * another factor's account with him.
     */
    V_66("66", "Current account"),

    /**
     * Document/message issued by a party (usually the buyer) to indicate that one or more
     * invoices or one or more credit notes are disputed for payment.
     */
    V_67("67", "Commercial dispute"),

    /**
     * Document/message issued by a factor to a seller or to another factor to indicate that
     * the rest of the amounts of one or more invoices uncollectable from buyers are charged
     * back to clear the invoice(s) off the ledger.
     */
    V_68("68", "Chargeback"),

    /**
     * Document/message issued by a factor to a seller or to another factor to reassign an
     * invoice or credit note previously assigned to him.
     */
    V_69("69", "Reassignment"),

    /**
     * Document message issued by a factor to indicate the movements of invoices, credit notes
     * and payments of a seller's account.
     */
    V_70("70", "Collateral account"),

    /**
     * Document/message issued by a creditor to a debtor to request payment of one or more
     * invoices past due.
     */
    V_71("71", "Request for payment"),

    /**
     * A message or document issuing permission to unship cargo.
     */
    V_72("72", "Unship permit"),

    /**
     * Transmission of one or more statistical definitions.
     */
    V_73("73", "Statistical definitions"),

    /**
     * Transmission of one or more items of data or data sets.
     */
    V_74("74", "Statistical data"),

    /**
     * Request for one or more items or data sets of statistical data.
     */
    V_75("75", "Request for statistical data"),

    /**
     * Document/message to provide split quantities and delivery dates referring to a previous
     * delivery instruction.
     */
    V_76("76", "Call-off delivery"),

    /**
     * Message covers information about the consignment status.
     */
    V_77("77", "Consignment status report"),

    /**
     * Advice of inventory movements.
     */
    V_78("78", "Inventory movement advice"),

    /**
     * Advice of stock on hand.
     */
    V_79("79", "Inventory status advice"),

    /**
     * Debit information related to a transaction for goods or services to the relevant party.
     */
    V_80("80", "Debit note related to goods or services"),

    /**
     * Document message used to provide credit information related to a transaction for goods
     * or services to the relevant party.
     */
    V_81("81", "Credit note related to goods or services"),

    /**
     * Document/message claiming payment for the supply of metered services (e.g., gas, electricity,
     * etc.) supplied to a fixed meter whose consumption is measured over a period of time.
     */
    V_82("82", "Metered services invoice"),

    /**
     * Document message for providing credit information related to financial adjustments to
     * the relevant party, e.g., bonuses.
     */
    V_83("83", "Credit note related to financial adjustments"),

    /**
     * Document/message for providing debit information related to financial adjustments to
     * the relevant party.
     */
    V_84("84", "Debit note related to financial adjustments"),

    /**
     * Message/document identifying a customs manifest. The document itemises a list of cargo
     * prepared by shipping companies from bills of landing and presented to customs for formal
     * report of cargo.
     */
    V_85("85", "Customs manifest"),

    /**
     * A document code to indicate that the message being transmitted identifies all short
     * and surplus cargoes off-loaded from a vessel at a specified discharging port.
     */
    V_86("86", "Vessel unpack report"),

    /**
     * A document code to indicate that the message being transmitted is summary manifest information
     * for general cargo.
     */
    V_87("87", "General cargo summary manifest report"),

    /**
     * A document code to indicate that the message being transmitted is a consignment unpack
     * report only.
     */
    V_88("88", "Consignment unpack report"),

    /**
     * Document or message issued by the competent authority in the exporting country evidencing
     * that meat or meat by-products comply with the requirements set by the importing country.
     */
    V_89("89", "Meat and meat by-products sanitary certificate"),

    /**
     * Document or message issued by the competent authority in the exporting country evidencing
     * that meat food products comply with the requirements set by the importing country.
     */
    V_90("90", "Meat food products sanitary certificate"),

    /**
     * Document or message issued by the competent authority in the exporting country evidencing
     * that poultry products comply with the requirements set by the importing country.
     */
    V_91("91", "Poultry sanitary certificate"),

    /**
     * Document or message issued by the competent authority in the exporting country evidencing
     * that horsemeat products comply with the requirements set by the importing country.
     */
    V_92("92", "Horsemeat sanitary certificate"),

    /**
     * Document or message issued by the competent authority in the exporting country evidencing
     * that casing products comply with the requirements set by the importing country.
     */
    V_93("93", "Casing sanitary certificate"),

    /**
     * Document or message issued by the competent authority in the exporting country evidencing
     * that pharmaceutical products comply with the requirements set by the importing country.
     */
    V_94("94", "Pharmaceutical sanitary certificate"),

    /**
     * Document or message issued by the competent authority in the exporting country evidencing
     * that inedible products comply with the requirements set by the importing country.
     */
    V_95("95", "Inedible sanitary certificate"),

    /**
     * Notification of impending arrival details for vessel.
     */
    V_96("96", "Impending arrival"),

    /**
     * Message reporting the means of transport used to carry goods or cargo.
     */
    V_97("97", "Means of transport advice"),

    /**
     * Message reporting the arrival details of goods or cargo.
     */
    V_98("98", "Arrival information"),

    /**
     * Message/document sent by the cargo handler indicating that the cargo has moved from
     * a Customs controlled premise.
     */
    V_99("99", "Cargo release notification"),

    /**
     * Certificate asserting that the goods have been submitted to the excise authorities before
     * departure from the exporting country or before delivery in case of import traffic.
     */
    V_100("100", "Excise certificate"),

    /**
     * An official document providing registration details.
     */
    V_101("101", "Registration document"),

    /**
     * Used to specify that the message is a tax notification.
     */
    V_102("102", "Tax notification"),

    /**
     * Report on the movement of containers or other items of transport equipment being exchanged,
     * establishing relevant rental periods.
     */
    V_103("103", "Transport equipment direct interchange report"),

    /**
     * Advice that containers or other items of transport equipment may be expected to be delivered
     * to a certain location.
     */
    V_104("104", "Transport equipment impending arrival advice"),

    /**
     * Document/message issued within an enterprise to initiate the purchase of articles, materials
     * or services required for the production or manufacture of goods to be offered for sale
     * or otherwise supplied to customers.
     */
    V_105("105", "Purchase order"),

    /**
     * Report of damaged items of transport equipment that have been returned.
     */
    V_106("106", "Transport equipment damage report"),

    /**
     * Advice providing estimates of transport equipment maintenance and repair costs.
     */
    V_107("107", "Transport equipment maintenance and repair work estimate advice"),

    /**
     * Instruction to release an item of empty transport equipment to a specified party or
     * parties.
     */
    V_108("108", "Transport equipment empty release instruction"),

    /**
     * Report on the inward movement of cargo, containers or other items of transport equipment
     * which have been delivered to a facility by an inland carrier.
     */
    V_109("109", "Transport movement gate in report"),

    /**
     * Document/message issued within an enterprise to initiate the manufacture of goods to
     * be offered for sale.
     */
    V_110("110", "Manufacturing instructions"),

    /**
     * Report on the outward movement of cargo, containers or other items of transport equipment
     * (either full or empty) which have been picked up by an inland carrier.
     */
    V_111("111", "Transport movement gate out report"),

    /**
     * Instruction to unpack specified cargo from specified containers or other items of transport
     * equipment.
     */
    V_112("112", "Transport equipment unpacking instruction"),

    /**
     * Report on the completion of unpacking specified containers or other items of transport
     * equipment.
     */
    V_113("113", "Transport equipment unpacking report"),

    /**
     * Request for confirmation that an item of transport equipment will be available for collection.
     */
    V_114("114", "Transport equipment pick-up availability request"),

    /**
     * Confirmation that an item of transport equipment is available for collection.
     */
    V_115("115", "Transport equipment pick-up availability confirmation"),

    /**
     * Report that an item of transport equipment has been collected.
     */
    V_116("116", "Transport equipment pick-up report"),

    /**
     * Report on the movement of containers or other items of transport within a facility.
     */
    V_117("117", "Transport equipment shift report"),

    /**
     * Instruction to unload specified cargo, containers or transport equipment from a means
     * of transport.
     */
    V_118("118", "Transport discharge instruction"),

    /**
     * Report on cargo, containers or transport equipment unloaded from a particular means
     * of transport.
     */
    V_119("119", "Transport discharge report"),

    /**
     * Document/message issued within an enterprise ordering the taking out of stock of goods.
     */
    V_120("120", "Stores requisition"),

    /**
     * Instruction to load cargo, containers or transport equipment onto a means of transport.
     */
    V_121("121", "Transport loading instruction"),

    /**
     * Report on completion of loading cargo, containers or other transport equipment onto
     * a means of transport.
     */
    V_122("122", "Transport loading report"),

    /**
     * Authorisation to have transport equipment repaired or to have maintenance performed.
     */
    V_123("123", "Transport equipment maintenance and repair work authorisation"),

    /**
     * Report of the departure of a means of transport from a particular facility.
     */
    V_124("124", "Transport departure report"),

    /**
     * Advice that an item or items of empty transport equipment are available for return.
     */
    V_125("125", "Transport empty equipment advice"),

    /**
     * Order to accept items of transport equipment which are to be delivered by an inland
     * carrier (rail, road or barge) to a specified facility.
     */
    V_126("126", "Transport equipment acceptance order"),

    /**
     * Instruction to perform a specified service or services on an item or items of transport
     * equipment.
     */
    V_127("127", "Transport equipment special service instruction"),

    /**
     * Report on the number of items of transport equipment stored at one or more locations.
     */
    V_128("128", "Transport equipment stock report"),

    /**
     * Order to release cargo or items of transport equipment to a specified party.
     */
    V_129("129", "Transport cargo release order"),

    /**
     * Document/message issued within an enterprise containing data about goods sold, to be
     * used as the basis for the preparation of an invoice.
     */
    V_130("130", "Invoicing data sheet"),

    /**
     * Instruction to pack cargo into a container or other item of transport equipment.
     */
    V_131("131", "Transport equipment packing instruction"),

    /**
     * Notification of customs clearance of cargo or items of transport equipment.
     */
    V_132("132", "Customs clearance notice"),

    /**
     * Notice specifying expiration of Customs documents relating to cargo or items of transport
     * equipment.
     */
    V_133("133", "Customs documents expiration notice"),

    /**
     * Request for transport equipment to be made available for hire.
     */
    V_134("134", "Transport equipment on-hire request"),

    /**
     * Order to release empty items of transport equipment for on-hire to a lessee, and authorising
     * collection by or on behalf of a specified party.
     */
    V_135("135", "Transport equipment on-hire order"),

    /**
     * Request to terminate the lease on an item of transport equipment at a specified time.
     */
    V_136("136", "Transport equipment off-hire request"),

    /**
     * Order to perform a survey on specified items of transport equipment.
     */
    V_137("137", "Transport equipment survey order"),

    /**
     * Response to an order to conduct a survey of transport equipment.
     */
    V_138("138", "Transport equipment survey order response"),

    /**
     * Survey report of specified items of transport equipment.
     */
    V_139("139", "Transport equipment survey report"),

    /**
     * Document/message within an enterprise giving instructions on how goods are to be packed.
     */
    V_140("140", "Packing instructions"),

    /**
     * A document and/or message advising of items which have to be booked to a financial account.
     */
    V_141("141", "Advising items to be booked to a financial account"),

    /**
     * Order to draw up an estimate of the costs of maintenance or repair of transport equipment.
     */
    V_142("142", "Transport equipment maintenance and repair work estimate order"),

    /**
     * Report of transport equipment which has been repaired or has had maintenance performed.
     */
    V_143("143", "Transport equipment maintenance and repair notice"),

    /**
     * Order to make available empty containers.
     */
    V_144("144", "Empty container disposition order"),

    /**
     * Order that the containers or cargo specified are to be discharged from a vessel.
     */
    V_145("145", "Cargo vessel discharge order"),

    /**
     * Order that specified cargo, containers or groups of containers are to be loaded in or
     * on a vessel.
     */
    V_146("146", "Cargo vessel loading order"),

    /**
     * One purchase order that contains the orders of two or more vendors and the associated
     * delivery points for each.
     */
    V_147("147", "Multidrop order"),

    /**
     * A document authorizing the bailing of goods.
     */
    V_148("148", "Bailment contract"),

    /**
     * A document indicating an agreement containing basic terms and conditions applicable
     * to future contracts between two parties.
     */
    V_149("149", "Basic agreement"),

    /**
     * Document/message giving instructions about the transport of goods within an enterprise.
     */
    V_150("150", "Internal transport order"),

    /**
     * A document indicating the granting of funds.
     */
    V_151("151", "Grant"),

    /**
     * A document indicating a contract calling for the indefinite deliveries of indefinite
     * quantities of goods.
     */
    V_152("152", "Indefinite delivery indefinite quantity contract"),

    /**
     * A document indicating a contract calling for indefinite deliveries of definite quantities.
     */
    V_153("153", "Indefinite delivery definite quantity contract"),

    /**
     * A document indicating a requirements contract that authorizes the filling of all purchase
     * requirements during a specified contract period.
     */
    V_154("154", "Requirements contract"),

    /**
     * A document indicating an order that tasks a contractor to perform a specified function.
     */
    V_155("155", "Task order"),

    /**
     * A document indicating a plan that identifies which items will be made and which items
     * will be bought.
     */
    V_156("156", "Make or buy plan"),

    /**
     * A document indicating a plan that identifies the manufacturer's subcontracting strategy
     * for a specific contract.
     */
    V_157("157", "Subcontractor plan"),

    /**
     * A document indicating a summary of cost data.
     */
    V_158("158", "Cost data summary"),

    /**
     * A document indicating cost and price data whose accuracy has been certified.
     */
    V_159("159", "Certified cost and price data"),

    /**
     * A document indicating a determination of the wages to be paid.
     */
    V_160("160", "Wage determination"),

    /**
     * A report to provide the status of funds applicable to the contract.
     */
    V_161("161", "Contract Funds Status Report (CFSR)"),

    /**
     * A certification as to the accuracy of inspection and test results.
     */
    V_162("162", "Certified inspection and test results"),

    /**
     * A report that is both an inspection report for materials and a receiving document.
     */
    V_163("163", "Material inspection and receiving report"),

    /**
     * A document indicating a specification used to purchase an item.
     */
    V_164("164", "Purchasing specification"),

    /**
     * A document indicating a bond that guarantees the payment of monies or a performance.
     */
    V_165("165", "Payment or performance bond"),

    /**
     * A document that indicates the specification contains the security and classification
     * requirements for a contract.
     */
    V_166("166", "Contract security classification specification"),

    /**
     * A document indicating the specification of how an item is to be manufactured.
     */
    V_167("167", "Manufacturing specification"),

    /**
     * A document certifying that more than 50 percent of the cost of an item is attributed
     * to US origin.
     */
    V_168("168", "Buy America certificate of compliance"),

    /**
     * Notice to return leased containers.
     */
    V_169("169", "Container off-hire notice"),

    /**
     * Order to accept cargo to be delivered by a carrier.
     */
    V_170("170", "Cargo acceptance order"),

    /**
     * Notice specifying the pick-up of released cargo or containers from a certain address.
     */
    V_171("171", "Pick-up notice"),

    /**
     * Document or message that authorises receiver to plan orders, based on information in
     * this message, and send these orders as suggestions to the sender.
     */
    V_172("172", "Authorisation to plan and suggest orders"),

    /**
     * Document or message that authorises receiver to plan and ship orders based on information
     * in this message.
     */
    V_173("173", "Authorisation to plan and ship orders"),

    /**
     * The document or message is a drawing.
     */
    V_174("174", "Drawing"),

    /**
     * A report identifying the cost performance on a contract at specified levels of the work
     * breakdown structure (format 2 - organizational categories).
     */
    V_175("175", "Cost Performance Report (CPR) format 2"),

    /**
     * A report providing the status of the cost and schedule applicable to a contract.
     */
    V_176("176", "Cost Schedule Status Report (CSSR)"),

    /**
     * A report identifying the cost performance on a contract including the current month's
     * values at specified levels of the work breakdown structure (format 1 - work breakdown
     * structure).
     */
    V_177("177", "Cost Performance Report (CPR) format 1"),

    /**
     * A report identifying the cost performance on a contract that summarizes changes to a
     * contract over a given reporting period with beginning and ending values (format 3 -
     * baseline).
     */
    V_178("178", "Cost Performance Report (CPR) format 3"),

    /**
     * A report identifying the cost performance on a contract including forecasts of labour
     * requirements for the remaining portion of the contract (format 4 - staffing).
     */
    V_179("179", "Cost Performance Report (CPR) format 4"),

    /**
     * A report identifying the cost performance on a contract that summarizes cost or schedule
     * variances (format 5 - explanations and problem analysis).
     */
    V_180("180", "Cost Performance Report (CPR) format 5"),

    /**
     * Document or message progressively issued by the container terminal operator in charge
     * of discharging a vessel identifying containers that have been discharged from a specific
     * vessel at that point in time.
     */
    V_181("181", "Progressive discharge report"),

    /**
     * Confirmation of a balance at an entry date.
     */
    V_182("182", "Balance confirmation"),

    /**
     * Order to unload goods from a container.
     */
    V_183("183", "Container stripping order"),

    /**
     * Order to stuff specified goods or consignments in a container.
     */
    V_184("184", "Container stuffing order"),

    /**
     * Declaration to the public authority upon arrival of the conveyance.
     */
    V_185("185", "Conveyance declaration (arrival)"),

    /**
     * Declaration to the public authority upon departure of the conveyance.
     */
    V_186("186", "Conveyance declaration (departure)"),

    /**
     * Combined declaration of arrival and departure to the public authority.
     */
    V_187("187", "Conveyance declaration (combined)"),

    /**
     * A project plan for recovery after a delay or problem resolution.
     */
    V_188("188", "Project recovery plan"),

    /**
     * A project plan for the production of goods.
     */
    V_189("189", "Project production plan"),

    /**
     * Documents/messages issued within an enterprise for the for the purpose of collection
     * of production and other internal statistics, and for other administration purposes.
     */
    V_190("190", "Statistical and other administrative internal documents"),

    /**
     * A high level, all encompassing master schedule of activities to complete a project.
     */
    V_191("191", "Project master schedule"),

    /**
     * A priced tender based upon an alternate specification.
     */
    V_192("192", "Priced alternate tender bill of quantity"),

    /**
     * An estimate based upon a detailed, quantity based specification (bill of quantity).
     */
    V_193("193", "Estimated priced bill of quantity"),

    /**
     * Document/message providing a draft bill of quantity, issued in an unpriced form.
     */
    V_194("194", "Draft bill of quantity"),

    /**
     * Instruction for the collection of the documentary credit.
     */
    V_195("195", "Documentary credit collection instruction"),

    /**
     * Request for an amendment of a documentary credit.
     */
    V_196("196", "Request for an amendment of a documentary credit"),

    /**
     * Documentary credit amendment information.
     */
    V_197("197", "Documentary credit amendment information"),

    /**
     * Advice of an amendment of a documentary credit.
     */
    V_198("198", "Advice of an amendment of a documentary credit"),

    /**
     * Response to an amendment of a documentary credit.
     */
    V_199("199", "Response to an amendment of a documentary credit"),

    /**
     * Provides information on documentary credit issuance.
     */
    V_200("200", "Documentary credit issuance information"),

    /**
     * Request to establish a direct payment valuation.
     */
    V_201("201", "Direct payment valuation request"),

    /**
     * Document/message addressed, for instance, by a general contractor to the owner, in order
     * that a direct payment be made to a subcontractor.
     */
    V_202("202", "Direct payment valuation"),

    /**
     * Document/message establishing a provisional payment valuation.
     */
    V_203("203", "Provisional payment valuation"),

    /**
     * Document/message establishing the financial elements of a situation of works.
     */
    V_204("204", "Payment valuation"),

    /**
     * Document/message providing a confirmed assessment, by quantity, of the completed work
     * for a construction contract.
     */
    V_205("205", "Quantity valuation"),

    /**
     * Document/message providing an initial assessment, by quantity, of the completed work
     * for a construction contract.
     */
    V_206("206", "Quantity valuation request"),

    /**
     * Document/message providing a formal specification identifying quantities and prices
     * that are the basis of a contract for a construction project. BOQ means: Bill of quantity.
     */
    V_207("207", "Contract bill of quantities - BOQ"),

    /**
     * Document/message providing a detailed, quantity based specification, issued in an unpriced
     * form to invite tender prices.
     */
    V_208("208", "Unpriced bill of quantity"),

    /**
     * Document/message providing a detailed, quantity based specification, updated with prices
     * to form a tender submission for a construction contract. BOQ means: Bill of quantity.
     */
    V_209("209", "Priced tender BOQ"),

    /**
     * Document/message issued by a party interested in the purchase of goods specified therein
     * and indicating particular, desirable conditions regarding delivery terms, etc., addressed
     * to a prospective supplier with a view to obtaining an offer.
     */
    V_210("210", "Enquiry"),

    /**
     * Document/message containing a provisional assessment in support of a request for payment
     * for completed work for a construction contract.
     */
    V_211("211", "Interim application for payment"),

    /**
     * Document/message in which the debtor expresses the intention to pay.
     */
    V_212("212", "Agreement to pay"),

    /**
     * The message is a request for financial cancellation.
     */
    V_213("213", "Request for financial cancellation"),

    /**
     * The message contains pre-authorised direct debit(s).
     */
    V_214("214", "Pre-authorised direct debit(s)"),

    /**
     * Document/message by means of which a buyer informs a seller that the buyer intends to
     * enter into contractual negotiations.
     */
    V_215("215", "Letter of intent"),

    /**
     * Document/message providing an approved detailed, quantity based specification (bill
     * of quantity), in an unpriced form.
     */
    V_216("216", "Approved unpriced bill of quantity"),

    /**
     * A payment valuation for unscheduled items.
     */
    V_217("217", "Payment valuation for unscheduled items"),

    /**
     * The final payment request of a series of payment requests submitted upon completion
     * of all the work.
     */
    V_218("218", "Final payment request based on completion of work"),

    /**
     * A request for payment for completed units.
     */
    V_219("219", "Payment request for completed units"),

    /**
     * Document/message by means of which a buyer initiates a transaction with a seller involving
     * the supply of goods or services as specified, according to conditions set out in an
     * offer, or otherwise known to the buyer.
     */
    V_220("220", "Order"),

    /**
     * Usage of document/message for general order purposes with later split into quantities
     * and delivery dates and maybe delivery locations.
     */
    V_221("221", "Blanket order"),

    /**
     * Document/message ordering the remainder of a production's batch.
     */
    V_222("222", "Spot order"),

    /**
     * Document/message for goods in leasing contracts.
     */
    V_223("223", "Lease order"),

    /**
     * Document/message for urgent ordering.
     */
    V_224("224", "Rush order"),

    /**
     * Document/message to order repair of goods.
     */
    V_225("225", "Repair order"),

    /**
     * Document/message to provide split quantities and delivery dates referring to a previous
     * blanket order.
     */
    V_226("226", "Call off order"),

    /**
     * Order to deliver goods into stock with agreement on payment when goods are sold out
     * of this stock.
     */
    V_227("227", "Consignment order"),

    /**
     * Document/message to order samples.
     */
    V_228("228", "Sample order"),

    /**
     * Document/message informing buyer or seller of the replacement of goods previously ordered.
     */
    V_229("229", "Swap order"),

    /**
     * Change to an purchase order already sent.
     */
    V_230("230", "Purchase order change request"),

    /**
     * Response to an purchase order already received.
     */
    V_231("231", "Purchase order response"),

    /**
     * Document/message for hiring human resources or renting goods or equipment.
     */
    V_232("232", "Hire order"),

    /**
     * Document/message to order spare parts.
     */
    V_233("233", "Spare parts order"),

    /**
     * A price/sales catalogue containing special prices which are valid only for a specified
     * period or under specified conditions.
     */
    V_234("234", "Campaign price/sales catalogue"),

    /**
     * Document or message issued by party identifying the containers for which they are responsible.
     */
    V_235("235", "Container list"),

    /**
     * A message which enables the transmission of delivery or product forecasting requirements.
     */
    V_236("236", "Delivery forecast"),

    /**
     * A document or message to order cross docking services.
     */
    V_237("237", "Cross docking services order"),

    /**
     * The message contains non-pre-authorised direct debit(s).
     */
    V_238("238", "Non-pre-authorised direct debit(s)"),

    /**
     * The message contains rejected direct debit(s).
     */
    V_239("239", "Rejected direct debit(s)"),

    /**
     * (1174) Document/message giving instruction regarding the delivery of goods.
     */
    V_240("240", "Delivery instructions"),

    /**
     * Usage of DELFOR-message.
     */
    V_241("241", "Delivery schedule"),

    /**
     * Usage of DELJIT-message.
     */
    V_242("242", "Delivery just-in-time"),

    /**
     * The message contains pre-authorised direct debit request(s).
     */
    V_243("243", "Pre-authorised direct debit request(s)"),

    /**
     * The message contains non-pre-authorised direct debit request(s).
     */
    V_244("244", "Non-pre-authorised direct debit request(s)"),

    /**
     * Document/message issued by a buyer releasing the despatch of goods after receipt of
     * the Ready for despatch advice from the seller.
     */
    V_245("245", "Delivery release"),

    /**
     * Settlement of a letter of credit.
     */
    V_246("246", "Settlement of a letter of credit"),

    /**
     * The message is a bank to bank funds transfer.
     */
    V_247("247", "Bank to bank funds transfer"),

    /**
     * The message contains customer payment order(s).
     */
    V_248("248", "Customer payment order(s)"),

    /**
     * The message contains low value payment order(s) only.
     */
    V_249("249", "Low value payment order(s)"),

    /**
     * Declaration regarding crew members aboard the conveyance.
     */
    V_250("250", "Crew list declaration"),

    /**
     * This is a request for information.
     */
    V_251("251", "Inquiry"),

    /**
     * A response to a previously sent banking status message.
     */
    V_252("252", "Response to previous banking status message"),

    /**
     * A high level, all encompassing master plan to complete a project.
     */
    V_253("253", "Project master plan"),

    /**
     * A plan for project work to be completed.
     */
    V_254("254", "Project plan"),

    /**
     * A schedule of project activities to be completed.
     */
    V_255("255", "Project schedule"),

    /**
     * Available resources for project planning purposes.
     */
    V_256("256", "Project planning available resources"),

    /**
     * Work calendar information for project planning purposes.
     */
    V_257("257", "Project planning calendar"),

    /**
     * An order to supply fixed quantities of products at fixed regular intervals.
     */
    V_258("258", "Standing order"),

    /**
     * A document detailing times and dates of events pertaining to a cargo movement.
     */
    V_259("259", "Cargo movement event log"),

    /**
     * An analysis of the cargo for a voyage.
     */
    V_260("260", "Cargo analysis voyage report"),

    /**
     * A document which indicates that the customer is claiming credit in a self billing environment.
     */
    V_261("261", "Self billed credit note"),

    /**
     * Credit note for goods and services that covers multiple transactions involving more
     * than one invoice.
     */
    V_262("262", "Consolidated credit note - goods and services"),

    /**
     * A message detailing statuses related to the adjustment of inventory.
     */
    V_263("263", "Inventory adjustment status report"),

    /**
     * Instruction to perform one or more different movements of transport equipment.
     */
    V_264("264", "Transport equipment movement instruction"),

    /**
     * Report on one or more different movements of transport equipment.
     */
    V_265("265", "Transport equipment movement report"),

    /**
     * Report on one or more changes of status associated with an item or items of transport
     * equipment.
     */
    V_266("266", "Transport equipment status change report"),

    /**
     * Certificate attesting that fumigation has been performed.
     */
    V_267("267", "Fumigation certificate"),

    /**
     * Certificate attesting to the quality, origin or appellation of wine.
     */
    V_268("268", "Wine certificate"),

    /**
     * Certificate attesting that wool is free from specified risks to human or animal health.
     */
    V_269("269", "Wool health certificate"),

    /**
     * Paper document attached to a consignment informing the receiving party about contents
     * of this consignment.
     */
    V_270("270", "Delivery note"),

    /**
     * Document/message specifying the distribution of goods in individual packages (in trade
     * environment the despatch advice message is used for the packing list).
     */
    V_271("271", "Packing list"),

    /**
     * Requesting a new code.
     */
    V_272("272", "New code request"),

    /**
     * Request a change to an existing code.
     */
    V_273("273", "Code change request"),

    /**
     * Requesting a new simple data element.
     */
    V_274("274", "Simple data element request"),

    /**
     * Request a change to an existing simple data element.
     */
    V_275("275", "Simple data element change request"),

    /**
     * Requesting a new composite data element.
     */
    V_276("276", "Composite data element request"),

    /**
     * Request a change to an existing composite data element.
     */
    V_277("277", "Composite data element change request"),

    /**
     * Request a new segment.
     */
    V_278("278", "Segment request"),

    /**
     * Requesting a change to an existing segment.
     */
    V_279("279", "Segment change request"),

    /**
     * Request for a new message (NMR).
     */
    V_280("280", "New message request"),

    /**
     * Requesting a Message in Development (MiD).
     */
    V_281("281", "Message in development request"),

    /**
     * Requesting a change to an existing message.
     */
    V_282("282", "Modification of existing message"),

    /**
     * Report of assigned tracking numbers.
     */
    V_283("283", "Tracking number assignment report"),

    /**
     * Document/message defining the contents of a user directory set or parts thereof.
     */
    V_284("284", "User directory definition"),

    /**
     * Requesting a United Nations Standard Message (UNSM).
     */
    V_285("285", "United Nations standard message request"),

    /**
     * Document/message defining the contents of a service directory set or parts thereof.
     */
    V_286("286", "Service directory definition"),

    /**
     * Message covers information about the status.
     */
    V_287("287", "Status report"),

    /**
     * Message to describe a Kanban schedule.
     */
    V_288("288", "Kanban schedule"),

    /**
     * A message to submit master data, a set of data that is rarely changed, to identify and
     * describe products a supplier offers to their (potential) customer or buyer.
     */
    V_289("289", "Product data message"),

    /**
     * A claim for parts and/or labour charges incurred .
     */
    V_290("290", "A claim for parts and/or labour charges"),

    /**
     * A message providing a response to a previously transmitted delivery schedule.
     */
    V_291("291", "Delivery schedule response"),

    /**
     * A message requesting a party to inspect items.
     */
    V_292("292", "Inspection request"),

    /**
     * A message informing a party of the results of an inspection.
     */
    V_293("293", "Inspection report"),

    /**
     * A message used by an application to acknowledge reception of a message and/or to report
     * any errors.
     */
    V_294("294", "Application acknowledgement and error report"),

    /**
     * An invoice which requests payment for the difference in price between an original invoice
     * and the result of the application of a price variation formula.
     */
    V_295("295", "Price variation invoice"),

    /**
     * A credit note which is issued against a price variation invoice.
     */
    V_296("296", "Credit note for price variation"),

    /**
     * A message instructing a party to collect goods.
     */
    V_297("297", "Instruction to collect"),

    /**
     * Listing of all details of dangerous goods carried.
     */
    V_298("298", "Dangerous goods list"),

    /**
     * Code specifying the continued validity of previously submitted registration information.
     */
    V_299("299", "Registration renewal"),

    /**
     * Code specifying the modification of previously submitted registration information.
     */
    V_300("300", "Registration change"),

    /**
     * Code specifying a response to an occurrence of a registration message.
     */
    V_301("301", "Response to registration"),

    /**
     * A document specifying the criterion and format for exchanging information in an electronic
     * data interchange syntax.
     */
    V_302("302", "Implementation guideline"),

    /**
     * Document/message is a request for transfer.
     */
    V_303("303", "Request for transfer"),

    /**
     * A report to convey cost performance data for a project or contract.
     */
    V_304("304", "Cost performance report"),

    /**
     * A message to inform a message issuer that a previously sent message has been received
     * by the addressee's application, or that a previously sent message has been rejected
     * by the addressee's application.
     */
    V_305("305", "Application error and acknowledgement"),

    /**
     * A financial statement for a cash pool.
     */
    V_306("306", "Cash pool financial statement"),

    /**
     * Message to describe a sequence of product delivery.
     */
    V_307("307", "Sequenced delivery schedule"),

    /**
     * A credit note sent to the party paying on behalf of a number of buyers.
     */
    V_308("308", "Delcredere credit note"),

    /**
     * Final discharge report by healthcare provider.
     */
    V_309("309", "Healthcare discharge report, final"),

    /**
     * (1332) Document/message which, with a view to concluding a contract, sets out the conditions
     * under which the goods are offered.
     */
    V_310("310", "Offer / quotation"),

    /**
     * Document/message requesting a quote on specified goods or services.
     */
    V_311("311", "Request for quote"),

    /**
     * Message providing acknowledgement information at the business application level concerning
     * the processing of a message.
     */
    V_312("312", "Acknowledgement message"),

    /**
     * Message indicating that a message was rejected due to errors encountered at the application
     * level.
     */
    V_313("313", "Application error message"),

    /**
     * A consolidated voyage summary which contains the information in a certificate of analysis,
     * a voyage analysis and a cargo movement time log for a voyage.
     */
    V_314("314", "Cargo movement voyage summary"),

    /**
     * (1296) Document/message evidencing an agreement between the seller and the buyer for
     * the supply of goods or services; its effects are equivalent to those of an order followed
     * by an acknowledgement of order.
     */
    V_315("315", "Contract"),

    /**
     * Document to apply for usage of berth or mooring facilities.
     */
    V_316("316", "Application for usage of berth or mooring facilities"),

    /**
     * Document to apply for designation of berthing places.
     */
    V_317("317", "Application for designation of berthing places"),

    /**
     * Document to apply for shifting from the designated place in port.
     */
    V_318("318", "Application for shifting from the designated place in port"),

    /**
     * Supplementary document to apply for cargo operation of dangerous goods.
     */
    V_319("319", "Supplementary document for application for cargo operation of dangerous goods"),

    /**
     * Document/message acknowledging an undertaking to fulfil an order and confirming conditions
     * or acceptance of conditions.
     */
    V_320("320", "Acknowledgement of order"),

    /**
     * Supplementary document to apply for transport of dangerous goods.
     */
    V_321("321", "Supplementary document for application for transport of dangerous goods"),

    /**
     * Payment effected by an Optical Character Reading (OCR) document.
     */
    V_322("322", "Optical Character Reading (OCR) payment"),

    /**
     * Preliminary sales report sent before all the information is available.
     */
    V_323("323", "Preliminary sales report"),

    /**
     * Official document specifying, for a given dangerous goods item, information such as
     * nature of hazard, protective devices, actions to be taken in case of accident, spillage
     * or fire and first aid to be given.
     */
    V_324("324", "Transport emergency card"),

    /**
     * Document/message serving as a preliminary invoice, containing - on the whole - the same
     * information as the final invoice, but not actually claiming payment.
     */
    V_325("325", "Proforma invoice"),

    /**
     * Document/message specifying details of an incomplete invoice.
     */
    V_326("326", "Partial invoice"),

    /**
     * Document/message describing instructions for operation.
     */
    V_327("327", "Operating instructions"),

    /**
     * Plates on goods identifying and describing an article.
     */
    V_328("328", "Name/product plate"),

    /**
     * The document or message contains a bordereau describing co-insurance ceding information.
     */
    V_329("329", "Co-insurance ceding bordereau"),

    /**
     * Document/message issued by a supplier requesting instructions from the buyer regarding
     * the details of the delivery of goods ordered.
     */
    V_330("330", "Request for delivery instructions"),

    /**
     * Commercial transaction (invoice) will include a packing list.
     */
    V_331("331", "Commercial invoice which includes a packing list"),

    /**
     * Document/message is for trade data.
     */
    V_332("332", "Trade data"),

    /**
     * Declaration provided to customs for cargo examination.
     */
    V_333("333", "Customs declaration for cargo examination"),

    /**
     * Alternate declaration provided to customs for cargo examination.
     */
    V_334("334", "Customs declaration for cargo examination, alternate"),

    /**
     * Document/message issued by a supplier to a carrier requesting space to be reserved for
     * a specified consignment, indicating desirable conveyance, despatch time, etc.
     */
    V_335("335", "Booking request"),

    /**
     * Document/message contains information regarding the crew list and conveyance.
     */
    V_336("336", "Customs crew and conveyance"),

    /**
     * Alternate Customs declaration summary with commercial transaction details.
     */
    V_337("337", "Customs summary declaration with commercial detail, alternate"),

    /**
     * A message reporting items which have been booked to a financial account.
     */
    V_338("338", "Items booked to a financial account report"),

    /**
     * A message reporting transactions which need further information from the receiver.
     */
    V_339("339", "Report of transactions which need further information from the receiver"),

    /**
     * (1121) Document/message advising details of cargo and exporter's requirements for its
     * physical movement.
     */
    V_340("340", "Shipping instructions"),

    /**
     * Document/message issued by a consignor in which he gives details of a consignment of
     * goods that enables an airline or its agent to prepare an air waybill.
     */
    V_341("341", "Shipper's letter of instructions (air)"),

    /**
     * A message reporting transactions for information only.
     */
    V_342("342", "Report of transactions for information only"),

    /**
     * Document/message giving instructions regarding local transport of goods, e.g. from the
     * premises of an enterprise to those of a carrier undertaking further transport.
     */
    V_343("343", "Cartage order (local transport)"),

    /**
     * A message giving additional information about the exchange of an EDI associated object.
     */
    V_344("344", "EDI associated object administration message"),

    /**
     * Document/message issued by a supplier informing a buyer that goods ordered are ready
     * for despatch.
     */
    V_345("345", "Ready for despatch advice"),

    /**
     * Sales report containing summaries for several earlier sent sales reports.
     */
    V_346("346", "Summary sales report"),

    /**
     * A message enquiring the status of previously sent orders.
     */
    V_347("347", "Order status enquiry"),

    /**
     * A message reporting the status of previously sent orders.
     */
    V_348("348", "Order status report"),

    /**
     * Document to declare inward and outward movement of a vessel.
     */
    V_349("349", "Declaration regarding the inward and outward movement of vessel"),

    /**
     * Document/message issued by a supplier initiating the despatch of goods to a buyer (consignee).
     */
    V_350("350", "Despatch order"),

    /**
     * Document/message by means of which the seller or consignor informs the consignee about
     * the despatch of goods.
     */
    V_351("351", "Despatch advice"),

    /**
     * Document to notify usage of berth or mooring facilities.
     */
    V_352("352", "Notification of usage of berth or mooring facilities"),

    /**
     * Document to apply for vessel's entering into port area in night-time.
     */
    V_353("353", "Application for vessel's entering into port area in night- time"),

    /**
     * Document to notify shifting from designated place in port once secured at the designated
     * place.
     */
    V_354("354", "Notification of emergency shifting from the designated place in port"),

    /**
     * Alternate Customs declaration summary without any commercial transaction details.
     */
    V_355("355", "Customs summary declaration without commercial detail, alternate"),

    /**
     * A document that guarantees performance.
     */
    V_356("356", "Performance bond"),

    /**
     * A document that guarantees the payment of monies.
     */
    V_357("357", "Payment bond"),

    /**
     * Preliminary discharge report by healthcare provider.
     */
    V_358("358", "Healthcare discharge report, preliminary"),

    /**
     * Document containing request for provision of a health service.
     */
    V_359("359", "Request for provision of a health service"),

    /**
     * Document/message requesting price conditions under which goods are offered.
     */
    V_360("360", "Request for price quote"),

    /**
     * Document/message confirming price conditions under which goods are offered.
     */
    V_361("361", "Price quote"),

    /**
     * Document/message confirming delivery conditions under which goods are offered.
     */
    V_362("362", "Delivery quote"),

    /**
     * Document/message confirming price and delivery conditions under which goods are offered.
     */
    V_363("363", "Price and delivery quote"),

    /**
     * Document/message confirming contractual price conditions under which goods are offered.
     */
    V_364("364", "Contract price quote"),

    /**
     * Document/message confirming contractual price conditions and contractual delivery conditions
     * under which goods are offered.
     */
    V_365("365", "Contract price and delivery quote"),

    /**
     * Document/message confirming price conditions under which goods are offered, provided
     * that they are sold to the end-customer specified on the quote.
     */
    V_366("366", "Price quote, specified end-customer"),

    /**
     * Document/message confirming price conditions and delivery conditions under which goods
     * are offered, provided that they are sold to the end-customer specified on the quote.
     */
    V_367("367", "Price and delivery quote, specified end-customer"),

    /**
     * Document/message from a supplier to a distributor confirming price conditions under
     * which goods can be sold by a distributor to the end-customer specified on the quote
     * with compensation for loss of inventory value.
     */
    V_368("368", "Price quote, ship and debit"),

    /**
     * Document/message from a supplier to a distributor confirming price conditions and delivery
     * conditions under which goods can be sold by a distributor to the end-customer specified
     * on the quote with compensation for loss of inventory value.
     */
    V_369("369", "Price and delivery quote, ship and debit"),

    /**
     * Document/message in which the party responsible for the issue of a set of trade documents
     * specifies the various recipients of originals and copies of these documents, with an
     * indication of the number of copies distributed to each of them.
     */
    V_370("370", "Advice of distribution of documents"),

    /**
     * Document containing a plan for provision of health service.
     */
    V_371("371", "Plan for provision of health service"),

    /**
     * Instructions for the dispensing and use of medicine or remedy.
     */
    V_372("372", "Prescription"),

    /**
     * Request to issue a prescription for medicine or remedy.
     */
    V_373("373", "Prescription request"),

    /**
     * Document containing information of products dispensed according to a prescription.
     */
    V_374("374", "Prescription dispensing report"),

    /**
     * (1109) Certificate providing confirmation that a consignment has been shipped.
     */
    V_375("375", "Certificate of shipment"),

    /**
     * A product inquiry which stands until it is cancelled.
     */
    V_376("376", "Standing inquiry on product information"),

    /**
     * Document/message providing data concerning the credit information of a party.
     */
    V_377("377", "Party credit information"),

    /**
     * Document/message providing data concerning the payment behaviour of a party.
     */
    V_378("378", "Party payment behaviour information"),

    /**
     * Message to request information about a metering point.
     */
    V_379("379", "Request for metering point information"),

    /**
     * (1334) Document/message claiming payment for goods or services supplied under conditions
     * agreed between seller and buyer.
     */
    V_380("380", "Commercial invoice"),

    /**
     * (1113) Document/message for providing credit information to the relevant party.
     */
    V_381("381", "Credit note"),

    /**
     * (1111) Document/message in which a seller specifies the amount of commission, the percentage
     * of the invoice amount, or some other basis for the calculation of the commission to
     * which a sales agent is entitled.
     */
    V_382("382", "Commission note"),

    /**
     * Document/message for providing debit information to the relevant party.
     */
    V_383("383", "Debit note"),

    /**
     * Commercial invoice that includes revised information differing from an earlier submission
     * of the same invoice.
     */
    V_384("384", "Corrected invoice"),

    /**
     * Commercial invoice that covers multiple transactions involving more than one vendor.
     */
    V_385("385", "Consolidated invoice"),

    /**
     * An invoice to pay amounts for goods and services in advance; these amounts will be subtracted
     * from the final invoice.
     */
    V_386("386", "Prepayment invoice"),

    /**
     * Document/message for invoicing the hiring of human resources or renting goods or equipment.
     */
    V_387("387", "Hire invoice"),

    /**
     * An invoice for tax purposes.
     */
    V_388("388", "Tax invoice"),

    /**
     * An invoice the invoicee is producing instead of the seller.
     */
    V_389("389", "Self-billed invoice"),

    /**
     * An invoice sent to the party paying for a number of buyers.
     */
    V_390("390", "Delcredere invoice"),

    /**
     * Response to a request for information about a metering point.
     */
    V_391("391", "Metering point information response"),

    /**
     * A notification of a change of supplier.
     */
    V_392("392", "Notification of change of supplier"),

    /**
     * Invoice assigned to a third party for collection.
     */
    V_393("393", "Factored invoice"),

    /**
     * Usage of INVOIC-message for goods in leasing contracts.
     */
    V_394("394", "Lease invoice"),

    /**
     * Commercial invoice that covers a transaction other than one involving a sale.
     */
    V_395("395", "Consignment invoice"),

    /**
     * Credit note related to assigned invoice(s).
     */
    V_396("396", "Factored credit note"),

    /**
     * A document providing a response to a previously sent commercial account summary message.
     */
    V_397("397", "Commercial account summary response"),

    /**
     * Document by means of which the supplier or consignor informs the buyer, consignee or
     * the distribution centre about the despatch of goods for cross docking.
     */
    V_398("398", "Cross docking despatch advice"),

    /**
     * Document by means of which the supplier or consignor informs the buyer, consignee or
     * the distribution centre about the despatch of goods for transshipment.
     */
    V_399("399", "Transshipment despatch advice"),

    /**
     * An order which falls outside the framework of an agreement.
     */
    V_400("400", "Exceptional order"),

    /**
     * An order requesting the supply of products packed according to the final delivery point
     * which will be moved across a dock in a distribution centre without further handling.
     */
    V_401("401", "Pre-packed cross docking order"),

    /**
     * An order requesting the supply of products which will be moved across a dock, de-consolidated
     * and re-consolidated according to the final delivery location requirements.
     */
    V_402("402", "Intermediate handling cross docking order"),

    /**
     * Information giving the various availabilities of a means of transportation.
     */
    V_403("403", "Means of transportation availability information"),

    /**
     * Information giving the various schedules of a means of transportation.
     */
    V_404("404", "Means of transportation schedule information"),

    /**
     * Notification regarding the delivery of transport equipment.
     */
    V_405("405", "Transport equipment delivery notice"),

    /**
     * Notification to the supplier regarding the termination of a contract.
     */
    V_406("406", "Notification to supplier of contract termination"),

    /**
     * Notification to the supplier about changes regarding a metering point.
     */
    V_407("407", "Notification to supplier of metering point changes"),

    /**
     * Notification about the change of a meter.
     */
    V_408("408", "Notification of meter change"),

    /**
     * Document/message containing instructions from a customer to his bank to pay an amount
     * in a specified currency to a nominated party in another country by a method either specified
     * (e.g. teletransmission, air mail) or left to the discretion of the bank.
     */
    V_409("409", "Instructions for bank transfer"),

    /**
     * Notification of the change of metering point identification.
     */
    V_410("410", "Notification of metering point identification change"),

    /**
     * The Utilities time series message is sent between responsible parties in a utilities
     * infrastructure for the purpose of reporting time series and connected technical and/or
     * administrative information.
     */
    V_411("411", "Utilities time series message"),

    /**
     * Application by a customer to his bank to issue a banker's draft stating the amount and
     * currency of the draft, the name of the payee and the place and country of payment.
     */
    V_412("412", "Application for banker's draft"),

    /**
     * Information about components in an infrastructure.
     */
    V_413("413", "Infrastructure condition"),

    /**
     * Acknowledgement of the change of supplier.
     */
    V_414("414", "Acknowledgement of change of supplier"),

    /**
     * Document/Message providing technical description and information of the crop production.
     */
    V_415("415", "Data Plot Sheet"),

    /**
     * Soil analysis document.
     */
    V_416("416", "Soil analysis"),

    /**
     * Farmyard manure analysis document.
     */
    V_417("417", "Farmyard manure analysis"),

    /**
     * Declaration, in accordance with the WCO Customs Data Model, to Customs concerning the
     * export of cargo carried by commercial means of transport over land, e.g. truck or train.
     */
    V_418("418", "WCO Cargo Report Export, Rail or Road"),

    /**
     * Declaration, in accordance with the WCO Customs Data Model, to Customs concerning the
     * export of cargo carried by commercial means of transport over water or through the air,
     * e.g. vessel or aircraft.
     */
    V_419("419", "WCO Cargo Report Export, Air or Maritime"),

    /**
     * Payment credit note effected by an Optical Character Reading (OCR) document.
     */
    V_420("420", "Optical Character Reading (OCR) payment credit note"),

    /**
     * Declaration, in accordance with the WCO Customs Data Model, to Customs concerning the
     * import of cargo carried by commercial means of transport over land, e.g. truck or train.
     */
    V_421("421", "WCO Cargo Report Import, Rail or Road"),

    /**
     * Declaration, in accordance with the WCO Customs Data Model, to Customs concerning the
     * import of cargo carried by commercial means of transport over water or through the air,
     * e.g. vessel or aircraft.
     */
    V_422("422", "WCO Cargo Report Import, Air or Maritime"),

    /**
     * Single step declaration, in accordance with the WCO Customs Data Model, to Customs by
     * which goods are declared for a Customs export procedure based on the 1999 Kyoto Convention.
     */
    V_423("423", "WCO one-step export declaration"),

    /**
     * First part of a simplified declaration, in accordance with the WCO Customs Data Model,
     * to Customs by which goods are declared for Customs export procedure based on the 1999
     * Kyoto Convention.
     */
    V_424("424", "WCO first step of two-step export declaration"),

    /**
     * Document/message whereby a bank advises that a collection has been paid, giving details
     * and methods of funds disposal.
     */
    V_425("425", "Collection payment advice"),

    /**
     * Document/message whereby a bank advises payment under a documentary credit.
     */
    V_426("426", "Documentary credit payment advice"),

    /**
     * Document/message whereby a bank advises acceptance under a documentary credit.
     */
    V_427("427", "Documentary credit acceptance advice"),

    /**
     * Document/message whereby a bank advises negotiation under a documentary credit.
     */
    V_428("428", "Documentary credit negotiation advice"),

    /**
     * Document/message whereby a customer requests his bank to issue a guarantee in favour
     * of a nominated party in another country, stating the amount and currency and the specific
     * conditions of the guarantee.
     */
    V_429("429", "Application for banker's guarantee"),

    /**
     * Document/message in which a bank undertakes to pay out a limited amount of money to
     * a designated party, on conditions stated therein (other than those laid down in the
     * Uniform Customs Practice).
     */
    V_430("430", "Banker's guarantee"),

    /**
     * Document/message in which a beneficiary of a documentary credit accepts responsibility
     * for non-compliance with the terms and conditions of the credit, and undertakes to refund
     * the money received under the credit, with interest and charges accrued.
     */
    V_431("431", "Documentary credit letter of indemnity"),

    /**
     * Notification to the grid operator regarding the termination of a contract.
     */
    V_432("432", "Notification to grid operator of contract termination"),

    /**
     * Notification to the grid operator about changes regarding a metering point.
     */
    V_433("433", "Notification to grid operator of metering point changes"),

    /**
     * Notification of a change of balance responsible entity.
     */
    V_434("434", "Notification of balance responsible entity change"),

    /**
     * Preadvice indicating a credit to happen in the future.
     */
    V_435("435", "Preadvice of a credit"),

    /**
     * Report on the profile of transport equipment.
     */
    V_436("436", "Transport equipment profile report"),

    /**
     * Document/message requesting price conditions and delivery conditions under which goods
     * are offered, provided that they are sold to the end-customer specified on the request
     * for quote.
     */
    V_437("437", "Request for price and delivery quote, specified end-user"),

    /**
     * Document/message from a distributor to a supplier requesting price conditions under
     * which goods can be sold by the distributor to the end-customer specified on the request
     * for quote with compensation for loss of inventory value.
     */
    V_438("438", "Request for price quote, ship and debit"),

    /**
     * Document/message from a distributor to a supplier requesting price conditions and delivery
     * conditions under which goods can be sold by the distributor to the end-customer specified
     * on the request for quote with compensation for loss of inventory value.
     */
    V_439("439", "Request for price and delivery quote, ship and debit"),

    /**
     * A list of delivery point addresses.
     */
    V_440("440", "Delivery point list."),

    /**
     * Document specifying the routes for transport between locations.
     */
    V_441("441", "Transport routing information"),

    /**
     * Document/message requesting delivery conditions under which goods are offered.
     */
    V_442("442", "Request for delivery quote"),

    /**
     * Document/message requesting price and delivery conditions under which goods are offered.
     */
    V_443("443", "Request for price and delivery quote"),

    /**
     * Document/message requesting contractual price conditions under which goods are offered.
     */
    V_444("444", "Request for contract price quote"),

    /**
     * Document/message requesting contractual price conditions and contractual delivery conditions
     * under which goods are offered.
     */
    V_445("445", "Request for contract price and delivery quote"),

    /**
     * Document/message requesting price conditions under which goods are offered, provided
     * that they are sold to the end-customer specified on the request for quote.
     */
    V_446("446", "Request for price quote, specified end-customer"),

    /**
     * Document/message whereby a bank is instructed (or requested) to handle financial and/or
     * commercial documents in order to obtain acceptance and/or payment, or to deliver documents
     * on such other terms and conditions as may be specified.
     */
    V_447("447", "Collection order"),

    /**
     * Document/message whereby a draft or similar instrument and/or commercial documents are
     * presented to a bank for acceptance, discounting, negotiation, payment or collection,
     * whether or not against a documentary credit.
     */
    V_448("448", "Documents presentation form"),

    /**
     * Message related to conducting a search for an identification match.
     */
    V_449("449", "Identification match"),

    /**
     * Document/message containing information needed to initiate the payment. It may cover
     * the financial settlement for one or more commercial trade transactions. A payment order
     * is an instruction to the ordered bank to arrange for the payment of one specified amount
     * to the beneficiary.
     */
    V_450("450", "Payment order"),

    /**
     * Document/message containing information needed to initiate the payment. It may cover
     * the financial settlement for several commercial trade transactions, which it is possible
     * to specify in a special payments detail part. It is an instruction to the ordered bank
     * to arrange for the payment of one specified amount to the beneficiary.
     */
    V_451("451", "Extended payment order"),

    /**
     * Document/message containing a payment order to debit one or more accounts and to credit
     * one or more beneficiaries.
     */
    V_452("452", "Multiple payment order"),

    /**
     * Message used to inform a supplier that delivered goods cannot be paid due to circumstances
     * which prevent payment.
     */
    V_453("453", "Notice that circumstances prevent payment of delivered goods"),

    /**
     * Document/message sent by an account servicing institution to one of its account owners,
     * to inform the account owner of an entry which has been or will be credited to its account
     * for a specified amount on the date indicated.
     */
    V_454("454", "Credit advice"),

    /**
     * Document/message sent by an account servicing institution to one of its account owners,
     * to inform the account owner of an entry that has been or will be credited to its account
     * for a specified amount on the date indicated. It provides extended commercial information
     * concerning the relevant remittance advice.
     */
    V_455("455", "Extended credit advice"),

    /**
     * Advice on a debit.
     */
    V_456("456", "Debit advice"),

    /**
     * Reversal of debit accounting entry by bank.
     */
    V_457("457", "Reversal of debit"),

    /**
     * Reversal of credit accounting entry by bank.
     */
    V_458("458", "Reversal of credit"),

    /**
     * The document is a ticket giving access to a travel service.
     */
    V_459("459", "Travel ticket"),

    /**
     * Document/message whereby a bank is requested to issue a documentary credit on the conditions
     * specified therein.
     */
    V_460("460", "Documentary credit application"),

    /**
     * The document is a credit, guarantee or charge card.
     */
    V_461("461", "Payment card"),

    /**
     * Document to advise that the goods ordered are ready for transshipment.
     */
    V_462("462", "Ready for transshipment despatch advice"),

    /**
     * Document by means of which the supplier or consignor informs the buyer, consignee or
     * distribution centre about the despatch of products packed according to the final delivery
     * point requirements which will be moved across a dock in a distribution centre without
     * further handling.
     */
    V_463("463", "Pre-packed cross docking despatch advice"),

    /**
     * Document by means of which the supplier or consignor informs the buyer, consignee or
     * the distribution centre about the despatch of products which will be moved across a
     * dock, de-consolidated and re-consolidated according to final delivery location requirements.
     */
    V_464("464", "Intermediate handling cross docking despatch advice"),

    /**
     * Document/message in which a bank states that it has issued a documentary credit under
     * which the beneficiary is to obtain payment, acceptance or negotiation on compliance
     * with certain terms and conditions and against presentation of stipulated documents and
     * such drafts as may be specified. The credit may or may not be confirmed by another bank.
     */
    V_465("465", "Documentary credit"),

    /**
     * Document/message issued by an advising bank in order to transmit a documentary credit
     * to a beneficiary, or to another advising bank.
     */
    V_466("466", "Documentary credit notification"),

    /**
     * Document/message whereby a bank advises that (part of) a documentary credit is being
     * or has been transferred in favour of a second beneficiary.
     */
    V_467("467", "Documentary credit transfer advice"),

    /**
     * Document/message whereby a bank advises that the terms and conditions of a documentary
     * credit have been amended.
     */
    V_468("468", "Documentary credit amendment notification"),

    /**
     * Document/message whereby a bank notifies a beneficiary of the details of an amendment
     * to the terms and conditions of a documentary credit.
     */
    V_469("469", "Documentary credit amendment"),

    /**
     * Document/message sent by a shipping agent to an authority for reporting information
     * on waste disposal.
     */
    V_470("470", "Waste disposal report"),

    /**
     * An invoice issued by a party who is out of the scope of tax regulations and shall not
     * collect tax on the invoice. The invoice should not contain tax details or information
     * about the party tax registrations.
     */
    @Deprecated
    V_480("480", "Invoice out of scope of tax"),

    /**
     * Document/message advising of the remittance of payment.
     */
    V_481("481", "Remittance advice"),

    /**
     * Document/message sent by a port authority to another port authority for reporting information
     * on waste disposal.
     */
    V_482("482", "Port authority waste disposal report"),

    /**
     * An endorsement on a passport or any other recognised travel document indicating that
     * it has been examined and found correct, especially as permitting the holder to enter
     * or leave a country.
     */
    V_483("483", "Visa"),

    /**
     * Document/message containing a direct debit request to credit one or more accounts and
     * to debit one or more debtors.
     */
    V_484("484", "Multiple direct debit request"),

    /**
     * Draft drawn in favour of a third party either by one bank on another bank, or by a branch
     * of a bank on its head office (or vice versa) or upon another branch of the same bank.
     * In either case, the draft should comply with the specifications laid down for cheques
     * in the country in which it is to be payable.
     */
    V_485("485", "Banker's draft"),

    /**
     * Document/message containing a direct debit to credit one or more accounts and to debit
     * one or more debtors.
     */
    V_486("486", "Multiple direct debit"),

    /**
     * Document or message issuing permission to disembark.
     */
    V_487("487", "Certificate of disembarkation permission"),

    /**
     * Document certifying that the object was free of rats when inspected and that it is exempt
     * from a deratting statement.
     */
    V_488("488", "Deratting exemption certificate"),

    /**
     * Order to connect a reefer container to a reefer point.
     */
    V_489("489", "Reefer connection order"),

    /**
     * Document/message, issued and signed in conformity with the applicable legislation, which
     * contains an unconditional order whereby the drawer directs the drawee to pay a definite
     * sum of money to the payee or to his order, on demand or at a definite time, against
     * the surrender of the document itself.
     */
    V_490("490", "Bill of exchange"),

    /**
     * Document/message, issued and signed in conformity with the applicable legislation, which
     * contains an unconditional promise whereby the maker undertakes to pay a definite sum
     * of money to the payee or to his order, on demand or at a definite time, against the
     * surrender of the document itself.
     */
    V_491("491", "Promissory note"),

    /**
     * Usage of STATAC-message.
     */
    V_493("493", "Statement of account message"),

    /**
     * Document/message ordering the direct delivery of goods/consignment from one means of
     * transport into another means of transport in one movement.
     */
    V_494("494", "Direct delivery (transport)"),

    /**
     * Second part of a simplified declaration, in accordance with the WCO Customs Data Model,
     * to Customs by which goods are declared for Customs export procedure based on the 1999
     * Kyoto Convention.
     */
    V_495("495", "WCO second step of two-step export declaration"),

    /**
     * Single step declaration, in accordance with the WCO Customs Data Model, to Customs by
     * which goods are declared for Customs import procedure based on the 1999 Kyoto Convention.
     */
    V_496("496", "WCO one-step import declaration"),

    /**
     * First part of a simplified declaration, in accordance with the WCO Customs Data Model,
     * to Customs by which goods are declared for Customs import procedure based on the 1999
     * Kyoto Convention.
     */
    V_497("497", "WCO first step of two-step import declaration"),

    /**
     * Second part of a simplified declaration, in accordance with the WCO Customs Data Model,
     * to Customs by which goods are declared for Customs import procedure based on the 1999
     * Kyoto Convention.
     */
    V_498("498", "WCO second step of two-step import declaration"),

    /**
     * Identification of the previous transport document.
     */
    V_499("499", "Previous transport document"),

    /**
     * An invoice produced by the buyer (invoicee) instead of the seller, which indicates that
     * the buyer has to pay amounts for goods and services in advance; these amounts will be
     * subtracted from the final invoice.
     */
    @Deprecated
    V_500("500", "Self billed prepayment invoice"),

    /**
     * An invoice assigned to a third party for collection, produced by the buyer (invoicee)
     * instead of the seller.
     */
    @Deprecated
    V_501("501", "Self billed factored invoice"),

    /**
     * An invoice assigned to a third party for collection, produced by the buyer (invoicee)
     * which indicates that the buyer is claiming credit.
     */
    @Deprecated
    V_502("502", "Self billed factored credit note"),

    /**
     * A document/message providing credit information to the relevant party which indicates
     * that the buyer has to pay amounts for goods and services in advance; these amounts will
     * be subtracted from the prepayment invoice.
     */
    @Deprecated
    V_503("503", "Prepayment credit note"),

    /**
     * Document/message issued to the insured certifying that insurance has been effected and
     * that a policy has been issued. Such a certificate for a particular cargo is primarily
     * used when good are insured under the terms of a floating or an open policy; at the request
     * of the insured it can be exchanged for a policy.
     */
    V_520("520", "Insurance certificate"),

    /**
     * A permit related to a transport document granting the transport of cargo under the conditions
     * as specifically required.
     */
    V_521("521", "Special requirements permit related to the transport of cargo"),

    /**
     * Dangerous Goods Notification for a vessel carrying liquid cargo in bulk.
     */
    V_522("522", "Dangerous Goods Notification for Tanker vessel"),

    /**
     * Dangerous Goods Notification for a vessel carrying cargo other than bulk liquid cargo.
     */
    V_523("523", "Dangerous Goods Notification for non-tanker vessel"),

    /**
     * Declaration, in accordance with the WCO Customs Data Model, to Customs regarding the
     * conveyance arriving in a Customs territory.
     */
    V_524("524", "WCO Conveyance Arrival Report"),

    /**
     * Declaration, in accordance with the WCO Customs Data Model, to Customs regarding the
     * conveyance departing a Customs territory.
     */
    V_525("525", "WCO Conveyance Departure Report"),

    /**
     * A document/message justifying an accounting entry.
     */
    V_526("526", "Accounting voucher"),

    /**
     * A document which indicates that the customer is claiming debit in a self billing environment.
     */
    V_527("527", "Self billed debit note"),

    /**
     * The official document used for military personnel on travel orders, substituting a passport.
     */
    V_528("528", "Military Identification Card"),

    /**
     * A permit to re-enter a country.
     */
    V_529("529", "Re-Entry Permit"),

    /**
     * Document/message issued by the insurer evidencing an agreement to insure and containing
     * the conditions of the agreement concluded whereby the insurer undertakes for a specific
     * fee to indemnify the insured for the losses arising out of the perils and accidents
     * specified in the contract.
     */
    V_530("530", "Insurance policy"),

    /**
     * Document identifying a refugee recognized by a country.
     */
    V_531("531", "Refugee Permit"),

    /**
     * Document/message for providing credit information to the relevant party.
     */
    V_532("532", "Forwarder’s credit note"),

    /**
     * To indicate that the document/message justifying an accounting entry is original.
     */
    V_533("533", "Original accounting voucher"),

    /**
     * To indicate that the document/message justifying an accounting entry is a copy.
     */
    V_534("534", "Copy accounting voucher"),

    /**
     * To indicate that the document/message justifying an accounting entry is pro-forma.
     */
    V_535("535", "Pro-forma accounting voucher"),

    /**
     * A certificate on ship security issued based on the International code for the Security
     * of Ships and of Port facilities (ISPS code).
     */
    V_536("536", "International Ship Security Certificate"),

    /**
     * An interim certificate on ship security issued basis under the International code for
     * the Security of Ships and of Port facilities (ISPS code).
     */
    V_537("537", "Interim International Ship Security Certificate"),

    /**
     * Certificate that guarantees quality manufacturing and processing of food products, medications,
     * cosmetics, etc.
     */
    V_538("538", "Good Manufacturing Practice (GMP) Certificate"),

    /**
     * An agreement between one or more contracting authorities and one or more economic operators,
     * the purpose of which is to establish the terms governing contracts to be awarded during
     * a given period, in particular with regard to price and, where appropriate, the quantity
     * envisaged.
     */
    V_539("539", "Framework Agreement"),

    /**
     * A document/message used when an insured reports to his insurer details of individual
     * shipments which are covered by an insurance contract - an open cover or a floating policy
     * - between the parties.
     */
    V_550("550", "Insurance declaration sheet (bordereau)"),

    /**
     * Offering of capacity for the transport of goods for a date and a route.
     */
    V_551("551", "Transport capacity offer"),

    /**
     * Ship Security Plan (SSP) is a document prepared in terms of the ISPS Code to contribute
     * to the prevention of illegal acts against the ship and its crew.
     */
    V_552("552", "Ship Security Plan"),

    /**
     * Document/message reporting invoice discrepancies indentified by the forwarder.
     */
    V_553("553", "Forwarder’s invoice discrepancy report"),

    /**
     * Offering of capacity to store goods.
     */
    V_554("554", "Storage capacity offer"),

    /**
     * Document/message issued by an insurer specifying the cost of an insurance which has
     * been effected and claiming payment therefore.
     */
    V_575("575", "Insurer's invoice"),

    /**
     * Request for capacity to store goods.
     */
    V_576("576", "Storage capacity request"),

    /**
     * Request for capacity for the transport of goods for a date and a route.
     */
    V_577("577", "Transport capacity request"),

    /**
     * Customs declaration for goods under the external Community/common transit procedure.
     * This applies to "non-Community goods" ("T1" under EU legislation and EC-EFTA "Transit
     * Convention").
     */
    V_578("578", "EU Customs declaration for External Community Transit (T1)"),

    /**
     * Customs declaration for goods under the internal Community/common transit procedure.
     * This applies to "Community goods" ("T2" under EU legislation and EC-EFTA "Transit Convention").
     */
    V_579("579", "EU Customs declaration for internal Community Transit (T2)"),

    /**
     * Document/message issued by an insurer (insurance broker, agent, etc.) to notify the
     * insured that his insurance have been carried out.
     */
    V_580("580", "Cover note"),

    /**
     * Declaration for goods under the internal Community transit procedure in the context
     * of trade between the "VAT" territory of EU Member States and EU territories where the
     * VAT rules do not apply, such as Canary islands, some French overseas territories, the
     * Channel islands and the Aaland islands, and between those territories. ("T2F" under
     * EU Legislation).
     */
    V_581("581", "EU Customs declaration for non-fiscal area internal Community Transit (T2F)"),

    /**
     * Customs declaration for goods under the internal Community transit procedure between
     * the Community and San Marino. ("T2SM" under EU Legislation).
     */
    V_582("582", "EU Customs declaration for internal transit to San Marino (T2SM)"),

    /**
     * Customs declaration for goods under the Community/common transit procedure for mixed
     * consignments (i.e. consignments that comprise goods of different statuses, like "T1"
     * and "T2") ("T" under EU Legislation).
     */
    V_583("583", "EU Customs declaration for mixed consignments (T)"),

    /**
     * Form establishing the Community status of goods ("T2L" under EU Legislation).
     */
    V_584("584", "EU Document for establishing the Community status of goods (T2L)"),

    /**
     * Form establishing the Community status of goods in the context of trade between the
     * "VAT" territory of EU Member States and EU territories where the VAT rules do not apply,
     * such as Canary islands, some French overseas territories, the Channel islands and the
     * Aaland islands, and between those territories ("T2LF" under EU Legislation).
     */
    V_585("585", "EU Document for establishing the Community status of goods for certain fiscal purposes (T2LF)"),

    /**
     * Form establishing the Community status of goods ("T2L" under European Legislation) in
     * the context of trade between the EU and San Marino. ("T2LSM" under EU Legislation).
     */
    V_586("586", "Document for establishing the Customs Status of goods for San Marino (T2LSM)"),

    /**
     * A Customs declaration in which goods move under cover of TIR Carnets.
     */
    V_587("587", "Customs declaration for TIR Carnet goods"),

    /**
     * A document reporting the security status and related information of a means of transport.
     */
    V_588("588", "Transport Means Security Report"),

    /**
     * A certificate verifying that meat has been produced from slaughter in accordance with
     * Islamic laws and practices.
     */
    V_589("589", "Halal Slaughtering Certificate"),

    /**
     * Document/message issued to a freight forwarder, giving instructions regarding the action
     * to be taken by the forwarder for the forwarding of goods described therein.
     */
    V_610("610", "Forwarding instructions"),

    /**
     * Document/message issued by a freight forwarder in an exporting country advising his
     * counterpart in an importing country about the forwarding of goods described therein.
     */
    V_621("621", "Forwarder's advice to import agent"),

    /**
     * Document/message issued by a freight forwarder informing an exporter of the action taken
     * in fulfillment of instructions received.
     */
    V_622("622", "Forwarder's advice to exporter"),

    /**
     * Invoice issued by a freight forwarder specifying services rendered and costs incurred
     * and claiming payment therefore.
     */
    V_623("623", "Forwarder's invoice"),

    /**
     * Non-negotiable document issued by a forwarder to certify that he has assumed control
     * of a specified consignment, with irrevocable instructions to send it to the consignee
     * indicated in the document or to hold it at his disposal. E.g. FIATA-FCR.
     */
    V_624("624", "Forwarder's certificate of receipt"),

    /**
     * A certificate verifying the heat treatment of the product is in conformance with international
     * standards to ensure the product’s healthiness and/or shows the mode of heat treatment
     * indicating the temperature and the amount of time the product or raw material used in
     * the product was treated (such as milk).
     */
    V_625("625", "Heat Treatment Certificate"),

    /**
     * A certificate used in the trade of endangered species in accordance with the CITES convention.
     */
    V_626("626", "Convention on International Trade in Endangered Species of Wild Fauna and Flora (CITES) Certificate"),

    /**
     * A certificate confirming that a specified product is free for sale in the country of
     * origin.
     */
    V_627("627", "Free Sale Certificate in the Country of Origin"),

    /**
     * Document/message issued by the competent body in accordance with transit regulations
     * in force, by which authorization is granted to a party to move articles under customs
     * procedure.
     */
    V_628("628", "Transit license"),

    /**
     * A certification that livestock or animal products, that are either imported or entering
     * free zones, are kept under health supervision for a time period determined by veterinary
     * quarantine instructions.
     */
    V_629("629", "Veterinary quarantine certificate"),

    /**
     * (1123) Document/message provided by the shipper or his agent to the carrier, multimodal
     * transport operator, terminal or other receiving authority, giving information about
     * export consignments offered for transport, and providing for the necessary receipts
     * and declarations of liability. Sometimes a multipurpose cargo handling document also
     * fulfilling the functions of document 632, 633, 650 and 655.
     */
    V_630("630", "Shipping note"),

    /**
     * Document/message issued by a forwarder acting as Warehouse Keeper acknowledging receipt
     * of goods placed in a warehouse, and stating or referring to the conditions which govern
     * the warehousing and the release of goods. The document contains detailed provisions
     * regarding the rights of holders-by-endorsement, transfer of ownership, etc. E.g. FIATA-FWR.
     */
    V_631("631", "Forwarder's warehouse receipt"),

    /**
     * Document/message to acknowledge the receipt of goods and in addition may indicate receiving
     * conditions.
     */
    V_632("632", "Goods receipt"),

    /**
     * Documents/messages specifying services rendered, storage and handling costs, demurrage
     * and other charges due to the owner of goods described therein.
     */
    V_633("633", "Port charges documents"),

    /**
     * A document legalized from a competent authority that shows the components of the product
     * (food additive, detergent, disinfectant and sanitizer).
     */
    V_634("634", "Certified list of ingredients"),

    /**
     * Negotiable receipt document, issued by a Warehouse Keeper to a person placing goods
     * in a warehouse and conferring title to the goods stored.
     */
    V_635("635", "Warehouse warrant"),

    /**
     * A document legalized from a competent authority that shows that the product has been
     * tested microbiologically and is free from any pathogens and fit for human consumption
     * and/or declares that the product is in compliance with sanitary and phytosanitary measures.
     */
    V_636("636", "Health certificate"),

    /**
     * A document that shows that the product (food additive, detergent, disinfectant and sanitizer)
     * is suitable to be used in the food industry.
     */
    V_637("637", "Food grade certificate"),

    /**
     * Certificate of inspection for the vessel stating its readiness and suitability for transporting
     * grains and legumes.
     */
    V_638("638", "Certificate of suitability for transport of grains and legumes"),

    /**
     * Inspection document shows that the container, the cooling devices and measured temperature
     * is in good working condition.
     */
    V_639("639", "Certificate of refrigerated transport equipment inspection"),

    /**
     * Document/message issued by a party entitled to authorize the release of goods specified
     * therein to a named consignee, to be retained by the custodian of the goods.
     */
    V_640("640", "Delivery order"),

    /**
     * A report of temperature readings over a period.
     */
    V_641("641", "Thermographic reading report"),

    /**
     * A certificate to verify readiness of a transport or transport area such as a reservoir
     * or hold to transport food items.
     */
    V_642("642", "Certificate of food item transport readiness"),

    /**
     * A document legalized from a competent authority that shows that the food packaging product
     * is safe to come into contact with food.
     */
    V_643("643", "Food packaging contact certificate"),

    /**
     * A document that shows the main structure that composes the packaging material.
     */
    V_644("644", "Packaging material composition report"),

    /**
     * A certification executed by the competent authority from country of exportation stating
     * the export price of the goods.
     */
    V_645("645", "Export price certificate"),

    /**
     * A certification executed by the competent authority from country of production stating
     * the price of the goods to the general public.
     */
    V_646("646", "Public price certificate"),

    /**
     * A document containing results from the study which determines the shelf life, namely
     * the time period of storage at a specified condition within which a drug substance or
     * drug product still meets its established specifications; its identity, strength, quality
     * and purity.
     */
    V_647("647", "Drug shelf life study report"),

    /**
     * A certification that the products have been treated in a way consistent with the standards
     * set by the World Organization for Animal Health (OIE).
     */
    V_648("648", "Certificate of compliance with standards of the World Organization for Animal Health (OIE)"),

    /**
     * A license granted by a competent authority to a production facility for manufacturing
     * specific products.
     */
    V_649("649", "Production facility license"),

    /**
     * Document/message issued by a cargo handling organization (port administration, terminal
     * operator, etc.) for the removal or other handling of goods under their care.
     */
    V_650("650", "Handling order"),

    /**
     * A license granted by a competent authority to a manufacturer for production of specific
     * products.
     */
    V_651("651", "Manufacturing license"),

    /**
     * An official letter issued by an import authority granted to the importer of goods from
     * a low risk country which allows the importer to place its products in the local market
     * with certain favorable considerations.
     */
    V_652("652", "Low risk country formal letter"),

    /**
     * Correspondence previously exchanged.
     */
    V_653("653", "Previous correspondence"),

    /**
     * A declaration to be presented to the competent authority when radioactive material moves
     * cross-border.
     */
    V_654("654", "Declaration for radioactive material"),

    /**
     * Document/message authorizing goods specified therein to be brought out of a fenced-in
     * port or terminal area.
     */
    V_655("655", "Gate pass"),

    /**
     * Document/message providing information on a resale.
     */
    V_656("656", "Resale information"),

    /**
     * A message/document consistent with the model for re-export phytosanitary certificates
     * of the IPPC, attesting that a consignment meets phytosanitary import requirements.
     */
    V_657("657", "Phytosanitary Re-export Certificate"),

    /**
     * A full bayplan containing all occupied and/or blocked stowage locations.
     */
    V_658("658", "Bayplan/stowage plan, full"),

    /**
     * A partial bayplan. containing only a selected part of the available stowage locations.
     */
    V_659("659", "Bayplan/stowage plan, partial"),

    /**
     * A document that provides production information such as the identification and number
     * of units of materials used and produced commonly related to manufacturing instructions,
     * purchase orders and other documents.
     */
    @Deprecated
    V_660("660", "Production Report"),

    /**
     * Non-negotiable document evidencing the contract for the transport of cargo.
     */
    V_700("700", "Waybill"),

    /**
     * Document/message evidencing a contract of carriage covering the movement of goods by
     * any mode of transport, or combination of modes, for national as well as international
     * transport, under any applicable international convention or national law and under the
     * conditions of carriage of any carrier or transport operator undertaking or arranging
     * the transport referred to in the document.
     */
    V_701("701", "Universal (multipurpose) transport document"),

    /**
     * Document/message issued by a carrier or a carrier's agent, acknowledging receipt for
     * carriage of goods specified therein on conditions stated or referred to in the document,
     * enabling the carrier to issue a transport document.
     */
    V_702("702", "Goods receipt, carriage"),

    /**
     * The document made out by an agent/consolidator which evidences the contract between
     * the shipper and the agent/consolidator for the arrangement of carriage of goods.
     */
    V_703("703", "House waybill"),

    /**
     * A bill of lading issued by the master of a vessel (in actuality the owner or charterer
     * of the vessel). It could cover a number of house bills.
     */
    V_704("704", "Master bill of lading"),

    /**
     * Negotiable document/message which evidences a contract of carriage by sea and the taking
     * over or loading of goods by carrier, and by which carrier undertakes to deliver goods
     * against surrender of the document. A provision in the document that goods are to be
     * delivered to the order of a named person, or to order, or to bearer, constitutes such
     * an undertaking.
     */
    V_705("705", "Bill of lading"),

    /**
     * The original of the bill of lading issued by a transport company. When issued by the
     * maritime industry it could signify ownership of the cargo.
     */
    V_706("706", "Bill of lading original"),

    /**
     * A copy of the bill of lading issued by a transport company.
     */
    V_707("707", "Bill of lading copy"),

    /**
     * Bill of lading indicating an empty container.
     */
    V_708("708", "Empty container bill"),

    /**
     * Document which evidences a transport of liquid bulk cargo.
     */
    V_709("709", "Tanker bill of lading"),

    /**
     * Non-negotiable document which evidences a contract for the carriage of goods by sea
     * and the taking over of the goods by the carrier, and by which the carrier undertakes
     * to deliver the goods to the consignee named in the document.
     */
    V_710("710", "Sea waybill"),

    /**
     * Negotiable transport document made out to a named person, to order or to bearer, signed
     * by the carrier and handed to the sender after receipt of the goods.
     */
    V_711("711", "Inland waterway bill of lading"),

    /**
     * Non-negotiable document which evidences a contract for the carriage of goods by sea
     * and the taking over or loading of the goods by the carrier, and by which the carrier
     * undertakes to deliver the goods to the consignee named in the document. E.g. Sea waybill.
     * Remark: Synonymous with "straight" or "non-negotiable Bill of lading" used in certain
     * countries, e.g. Canada.
     */
    V_712("712", "Non-negotiable maritime transport document (generic)"),

    /**
     * Document/message issued by a ship's officer to acknowledge that a specified consignment
     * has been received on board a vessel, and the apparent condition of the goods; enabling
     * the carrier to issue a Bill of lading.
     */
    V_713("713", "Mate's receipt"),

    /**
     * The bill of lading issued not by the carrier but by the freight forwarder/consolidator
     * known by the carrier.
     */
    V_714("714", "House bill of lading"),

    /**
     * Document/message issued by a commercial party or a bank of an insurance company accepting
     * responsibility to the beneficiary of the indemnity in accordance with the terms thereof.
     */
    V_715("715", "Letter of indemnity for non-surrender of bill of lading"),

    /**
     * Non-negotiable document issued by a freight forwarder evidencing a contract for the
     * carriage of goods by sea and the taking over or loading of the goods by the freight
     * forwarder, and by which the freight forwarder undertakes to deliver the goods to the
     * consignee named in the document.
     */
    V_716("716", "Forwarder's bill of lading"),

    /**
     * A document authorizing residence.
     */
    V_717("717", "Residence permit"),

    /**
     * A national identity document issued to professional seamen that contains a record of
     * their rank and service career.
     */
    V_718("718", "Seaman’s book"),

    /**
     * Document/message providing agreed textual information.
     */
    V_719("719", "General message"),

    /**
     * Transport document constituting a contract for the carriage of goods between the sender
     * and the carrier (the railway). For international rail traffic, this document must conform
     * to the model prescribed by the international conventions concerning carriage of goods
     * by rail, e.g. CIM Convention, SMGS Convention.
     */
    V_720("720", "Rail consignment note (generic term)"),

    /**
     * Document/message responding to a previously received Product Data document/message.
     */
    V_721("721", "Product data response"),

    /**
     * Accounting document, one copy of which is drawn up for each consignment note; it accompanies
     * the consignment over the whole route and is a rail transport document.
     */
    V_722("722", "Road list-SMGS"),

    /**
     * Document/message which gives right to the owner to exert all functions normally transferred
     * to a guard in a train by which an escorted consignment is transported.
     */
    V_723("723", "Escort official recognition"),

    /**
     * Fictitious transport document regarding a previous transport, enabling a carrier's agent
     * to give to another carrier's agent (in a different country) the possibility to collect
     * charges relating to the original transport (rail environment).
     */
    V_724("724", "Recharging document"),

    /**
     * Document/message providing details of an order which has been raised by a manufacturer.
     */
    V_725("725", "Manufacturer raised order"),

    /**
     * Document/message providing details of a consignment order which has been raised by a
     * manufacturer.
     */
    V_726("726", "Manufacturer raised consignment order"),

    /**
     * A price/sales catalogue message containing no commercial information, such as prices,
     * terms or conditions.
     */
    V_727("727", "Price/sales catalogue not containing commercial information"),

    /**
     * A price/sales catalogue message containing only commercial terms or conditions data.
     */
    V_728("728", "Price/sales catalogue containing commercial information"),

    /**
     * Document/message by means of which the buyer informs the seller about the despatch of
     * returned goods.
     */
    V_729("729", "Returns advice"),

    /**
     * Transport document/message which evidences a contract between a carrier and a sender
     * for the carriage of goods by road (generic term). Remark: For international road traffic,
     * this document must contain at least the particulars prescribed by the convention on
     * the contract for the international carriage of goods by road (CMR).
     */
    V_730("730", "Road consignment note"),

    /**
     * A message enabling the transmission of commercial data concerning payments made and
     * outstanding items on an account over a period of time.
     */
    V_731("731", "Commercial account summary"),

    /**
     * A message by which a party announces to another party details of goods for return due
     * to specified reasons (e.g. returns for repair, returns because of damage, etc).
     */
    V_732("732", "Announcement for returns"),

    /**
     * A message by which a party informs another party whether and how goods shall be returned.
     */
    V_733("733", "Instruction for returns"),

    /**
     * A message enabling companies to exchange or report electronically, basic sales forecast
     * data related to products or services, including the corresponding location, time period,
     * product identification, pricing and quantity information. It enables the recip.
     */
    V_734("734", "Sales forecast report"),

    /**
     * A message enabling companies to exchange or report electronically, basic sales data
     * related to products or services, including the corresponding location, time period,
     * product identification, pricing and quantity information. It enables the recipient to
     * p.
     */
    V_735("735", "Sales data report"),

    /**
     * A product inquiry which stands until it is cancelled. It requests not only the updates
     * since last time, but always the complete product information of a data supplier. This
     * means that within the standing request every time a complete download of the respe.
     */
    V_736("736", "Standing inquiry on complete product information"),

    /**
     * A message by which a consignee provides for a carrier proof of delivery of a consignment.
     */
    V_737("737", "Proof of delivery"),

    /**
     * A message from a party to a warehouse, distribution centre, or logistics service provider
     * identifying the handling services and where required the movement of specified goods,
     * limited to warehouses within the jurisdiction of the distribution centre or log.
     */
    V_738("738", "Cargo/goods handling and movement message"),

    /**
     * Document/message providing metered consumption details supporiting an invoice.
     */
    V_739("739", "Metered services consumption report supporting an invoice"),

    /**
     * Document/message made out by or on behalf of the shipper which evidences the contract
     * between the shipper and carrier(s) for carriage of goods over routes of the carrier(s)
     * and which is identified by the airline prefix issuing the document plus a serial (IATA).
     */
    V_740("740", "Air waybill"),

    /**
     * Document/message made out by or on behalf of the agent/consolidator which evidences
     * the contract between the agent/consolidator and carrier(s) for carriage of goods over
     * routes of the carrier(s) for a consignment consisting of goods originated by more than
     * one shipper (IATA).
     */
    V_741("741", "Master air waybill"),

    /**
     * Document/message providing metered consumption details.
     */
    V_742("742", "Metered services consumption report"),

    /**
     * A temporary air waybill which contains only limited information because of the absence
     * of the original.
     */
    V_743("743", "Substitute air waybill"),

    /**
     * Declaration to Customs regarding the personal effects of crew members aboard the conveyance;
     * equivalent to IMO FAL 4.
     */
    V_744("744", "Crew's effects declaration"),

    /**
     * Declaration to Customs regarding passengers aboard the conveyance; equivalent to IMO
     * FAL 6.
     */
    V_745("745", "Passenger list"),

    /**
     * Document/message created by the consignor or by the departure station, joined to the
     * transport or sent to the consignee, giving the possibility to the consignee or the arrival
     * station to attest the delivery of the goods. The document must be returned to the consignor
     * or to the departure station.
     */
    V_746("746", "Delivery notice (rail transport)"),

    /**
     * A message sent by a party (usually an employer or its representative) to a service providing
     * organisation, to detail payroll deductions paid on behalf of its employees to the service
     * providing organisation.
     */
    V_747("747", "Payroll deductions advice"),

    /**
     * Document/message by means of which the supplier informs the buyer about the despatch
     * of goods ordered on consignment (goods to be delivered into stock with agreement on
     * payment when goods are sold out of this stock).
     */
    V_748("748", "Consignment despatch advice"),

    /**
     * Message containing information regarding gross mass verification of transport equipment.
     */
    V_749("749", "Transport equipment gross mass verification message"),

    /**
     * Document/message which, according to Article 106 of the "Agreement concerning Postal
     * Parcels" under the UPU convention, is to accompany post parcels.
     */
    V_750("750", "Despatch note (post parcels)"),

    /**
     * A document / message containing accounting related information such as monetary summations,
     * seller id and VAT information. This may not be a complete invoice according to legal
     * requirements. For instance the line item information might be excluded.
     */
    V_751("751", "Invoice information for accounting purposes"),

    /**
     * Document/message issued by a competent body certifying the phytosanitary status of plants
     * or plant products for international trade.
     */
    V_752("752", "Plant Passport"),

    /**
     * Document/message issued by a competent body certifying sustainability.
     */
    V_753("753", "Certificate of sustainability"),

    /**
     * A document/message used by a buyer to define the procurement procedure and request suppliers
     * to participate.
     */
    V_754("754", "Call for tender"),

    /**
     * A document/message used by a buyer to define the procurement procedure and request specific
     * suppliers to participate.
     */
    V_755("755", "Invitation to tender"),

    /**
     * A document/message requesting a self-declaration from the supplier, providing preliminary
     * evidence during the tendering phase.
     */
    V_756("756", "European Single Procurement Document request"),

    /**
     * A document/message requesting information regarding pricing and catalogue details for
     * goods and/or services to be offered as part of a tender.
     */
    V_757("757", "Tendering price/sales catalogue request"),

    /**
     * A document/message used by a supplier to bid in a procurement procedure.
     */
    V_758("758", "Tender"),

    /**
     * A document/message containing a self-declaration by the supplier, providing preliminary
     * evidence during the tendering phase.
     */
    V_759("759", "European Single Procurement Document"),

    /**
     * A transport document used when more than one mode of transportation is involved in the
     * movement of cargo. It is a contract of carriage and receipt of the cargo for a multimodal
     * transport. It indicates the place where the responsible transport company in the move
     * takes responsibility for the cargo, the place where the responsibility of this transport
     * company in the move ends and the conveyances involved.
     */
    V_760("760", "Multimodal/combined transport document (generic)"),

    /**
     * Bill of lading which evidences a contract of carriage from one place to another in separate
     * stages of which at least one stage is a sea transit, and by which the issuing carrier
     * accepts responsibility for the carriage as set forth in the through bill of lading.
     */
    V_761("761", "Through bill of lading"),

    /**
     * A document/message providing information regarding pricing and catalogue details for
     * goods and/or services to be offered as part of a tender.
     */
    V_762("762", "Tendering price/sales catalogue"),

    /**
     * Negotiable document/message issued by a forwarder to certify that he has taken charge
     * of a specified consignment for despatch and delivery in accordance with the consignor's
     * instructions, as indicated in the document, and that he accepts responsibility for delivery
     * of the goods to the holder of the document through the intermediary of a delivery agent
     * of his choice. E.g. FIATA-FCT.
     */
    V_763("763", "Forwarder's certificate of transport"),

    /**
     * Negotiable or non-negotiable document evidencing a contract for the performance and/or
     * procurement of performance of combined transport of goods and bearing on its face either
     * the heading "Negotiable combined transport document issued subject to Uniform Rules
     * for a Combined Transport Document (ICC Brochure No. 298)" or the heading "Non-negotiable
     * Combined Transport Document issued subject to Uniform Rules for a Combined Transport
     * Document (ICC Brochure No. 298)".
     */
    V_764("764", "Combined transport document (generic)"),

    /**
     * Document/message which evidences a multimodal transport contract, the taking in charge
     * of the goods by the multimodal transport operator, and an undertaking by him to deliver
     * the goods in accordance with the terms of the contract. (International Convention on
     * Multimodal Transport of Goods).
     */
    V_765("765", "Multimodal transport document (generic)"),

    /**
     * Document which evidences a multimodal transport contract, the taking in charge of the
     * goods by the multimodal transport operator, and an undertaking by him to deliver the
     * goods in accordance with the terms of the contract.
     */
    V_766("766", "Combined transport bill of lading/multimodal bill of lading"),

    /**
     * Document/message confirming a receipt to the sending party.
     */
    V_767("767", "Acknowledgment of receipt"),

    /**
     * Document which confirms the civil status of a person.
     */
    V_768("768", "Civil status document"),

    /**
     * Document reporting advice.
     */
    V_769("769", "Advice report"),

    /**
     * Document/message issued by a carrier to confirm that space has been reserved for a consignment
     * in means of transport.
     */
    V_770("770", "Booking confirmation"),

    /**
     * Document which is a binding offer from one party to another.
     */
    V_771("771", "Binding offer"),

    /**
     * Document which is a binding agreement from the customer for a contract, such as an insurance
     * contract.
     */
    V_772("772", "Binding customer agreement for contract"),

    /**
     * Document confirming that insurance coverage is granted.
     */
    V_773("773", "Coverage confirmation note"),

    /**
     * Document specifying general terms and conditions.
     */
    V_774("774", "General terms and conditions"),

    /**
     * Instructions for release or delivery of goods.
     */
    V_775("775", "Calling forward notice"),

    /**
     * Document specifying the clauses applying to a contract.
     */
    V_776("776", "Contract clauses"),

    /**
     * Document specifying the individual conditions or clauses applying to a specific contract.
     */
    V_777("777", "Specific contract conditions"),

    /**
     * Document stating the rules of a group insurance contract.
     */
    V_778("778", "Group insurance rules"),

    /**
     * Document consisting of a series of questions.
     */
    V_779("779", "Questionnaire"),

    /**
     * Document/message issued by a transport operation specifying freight costs and charges
     * incurred for a transport operation and stating conditions of payment.
     */
    V_780("780", "Freight invoice"),

    /**
     * Notification from the carrier to the consignee in writing, by telephone or by any other
     * means (express letter, message, telegram, etc.) informing him that a consignment addressed
     * to him is being or will shortly be held at his disposal at a specified point in the
     * place of destination.
     */
    V_781("781", "Arrival notice (goods)"),

    /**
     * Request made by the carrier to the sender, or, as the case may be, the consignee, for
     * instructions as to the disposal of the consignment when circumstances prevent delivery
     * and the return of the goods has not been requested by the consignor in the transport
     * document.
     */
    V_782("782", "Notice of circumstances preventing delivery (goods)"),

    /**
     * Request made by the carrier to the sender, or, the consignee as the case may be, for
     * instructions as to the disposal of the goods when circumstances prevent transport before
     * departure or en route, after acceptance of the consignment concerned.
     */
    V_783("783", "Notice of circumstances preventing transport (goods)"),

    /**
     * Notification in writing, sent by the carrier to the sender, to inform him at his request
     * of the actual date of delivery of the goods.
     */
    V_784("784", "Delivery notice (goods)"),

    /**
     * Listing of goods comprising the cargo carried in a means of transport or in a transport-unit.
     * The cargo manifest gives the commercial particulars of the goods, such as transport
     * document numbers, consignors, consignees, shipping marks, number and kind of packages
     * and descriptions and quantities of the goods.
     */
    V_785("785", "Cargo manifest"),

    /**
     * Document/message containing the same information as a cargo manifest, and additional
     * details on freight amounts, charges, etc.
     */
    V_786("786", "Freight manifest"),

    /**
     * Document/message used in road transport, listing the cargo carried on a road vehicle,
     * often referring to appended copies of Road consignment note.
     */
    V_787("787", "Bordereau"),

    /**
     * Document/message specifying the contents of particular freight containers or other transport
     * units, prepared by the party responsible for their loading into the container or unit.
     */
    V_788("788", "Container manifest (unit packing list)"),

    /**
     * Document used by the rail organization to indicate freight charges or additional charges
     * in each case where the departure station is not able to calculate the charges for the
     * total voyage (e.g. tariff not yet updated, part of voyage not covered by the tariff).
     * This document must be considered as joined to the transport.
     */
    V_789("789", "Charges note"),

    /**
     * (1030) Document that is joined to the transport or sent by separate means, giving to
     * the departure rail organization the proof that the cash-on delivery amount has been
     * encashed by the arrival rail organization before reimbursement of the consignor.
     */
    V_790("790", "Advice of collection"),

    /**
     * Document certifying a ship's safety to a specified date.
     */
    V_791("791", "Safety of ship certificate"),

    /**
     * Document certifying the safety of a ship's radio facilities to a specified date.
     */
    V_792("792", "Safety of radio certificate"),

    /**
     * Document certifying the safety of a ship's equipment to a specified date.
     */
    V_793("793", "Safety of equipment certificate"),

    /**
     * Document declaring a ship owner's liability for oil propelling or carried on a vessel.
     */
    V_794("794", "Civil liability for oil certificate"),

    /**
     * Document specifying the limit of a ship's legal submersion under various conditions.
     */
    V_795("795", "Loadline document"),

    /**
     * Document certifying that a ship is free of rats, valid to a specified date.
     */
    V_796("796", "Derat document"),

    /**
     * Document certifying the health condition on board a vessel, valid to a specified date.
     */
    V_797("797", "Maritime declaration of health"),

    /**
     * Official certificate stating the vessel's registry.
     */
    V_798("798", "Certificate of registry"),

    /**
     * Declaration to Customs regarding the contents of the ship's stores (equivalent to IMO
     * FAL 3) i.e. goods intended for consumption by passengers/crew on board vessels, aircraft
     * or trains, whether or not sold or landed; goods necessary for operation/maintenance
     * of conveyance, including fuel/lubricants, excluding spare parts/equipment (IMO).
     */
    V_799("799", "Ship's stores declaration"),

    /**
     * Application for a permit issued by a government authority permitting exportation of
     * a specified commodity subject to specified conditions as quantity, country of destination,
     * etc.
     */
    V_810("810", "Export licence, application for"),

    /**
     * Permit issued by a government authority permitting exportation of a specified commodity
     * subject to specified conditions as quantity, country of destination, etc. Synonym: Embargo
     * permit.
     */
    V_811("811", "Export licence"),

    /**
     * Document/message completed by an exporter/seller as a means whereby the competent body
     * may control that the amount of foreign exchange accrued from a trade transaction is
     * repatriated in accordance with the conditions of payment and exchange control regulations
     * in force.
     */
    V_812("812", "Exchange control declaration, export"),

    /**
     * Declaration document to identify the final beneficiary of an asset.
     */
    V_813("813", "Declaration of final beneficiary"),

    /**
     * Statement regarding the Foreign Account Tax Compliance Act (FATCA) of the United States
     * of America.
     */
    V_814("814", "US, FATCA statement"),

    /**
     * Document reporting (e.g. annually) to the insured the actual details of an insurance
     * contract.
     */
    V_815("815", "Insured status report"),

    /**
     * Information document for the group pension commitment to an individual person.
     */
    V_816("816", "Group pension commitment information"),

    /**
     * Document notifying a claim.
     */
    V_817("817", "Claim notification"),

    /**
     * Document reporting an assessment.
     */
    V_818("818", "Assessment report"),

    /**
     * Document specifying the value of a loss.
     */
    V_819("819", "Loss statement"),

    /**
     * European community transit declaration.
     */
    V_820("820", "Despatch note model T"),

    /**
     * Transit declaration for goods circulating under internal community transit procedures
     * (between European Union (EU) countries).
     */
    V_821("821", "Despatch note model T1"),

    /**
     * Ascertainment that the declared goods were originally produced in an European Union
     * (EU) country.
     */
    V_822("822", "Despatch note model T2"),

    /**
     * Control document (export declaration) used particularly in case of re-sending without
     * use with only VAT collection, refusal, unconformity with contract etc.
     */
    V_823("823", "Control document T5"),

    /**
     * Rail consignment note prepared by the consignor for the facilitation of an eventual
     * return to the origin of the goods.
     */
    V_824("824", "Re-sending consignment note"),

    /**
     * Ascertainment that the declared goods were originally produced in an European Union
     * (EU) country. May only be used for goods that are loaded on one single means of transport
     * in one single departure point for one single delivery point.
     */
    V_825("825", "Despatch note model T2L"),

    /**
     * Document certifying the guarantee of the document issuer that he will pay for costs
     * of the addressee, e.g. the costs for repairing a vehicle.
     */
    V_826("826", "Guarantee of cost acceptance"),

    /**
     * Document reporting the closing of a claim file.
     */
    V_827("827", "Close of claim"),

    /**
     * Document stating the refusal of a claim.
     */
    V_828("828", "Refusal of claim"),

    /**
     * Document reporting a valuation.
     */
    V_829("829", "Valuation report"),

    /**
     * Document/message by which goods are declared for export Customs clearance, conforming
     * to the layout key set out at Appendix I to Annex C.1 concerning outright exportation
     * to the Kyoto convention (CCC). Within a Customs union, "for despatch" may have the same
     * meaning as "for exportation".
     */
    V_830("830", "Goods declaration for exportation"),

    /**
     * Document which certifies the history of claims.
     */
    V_831("831", "Claim history certificate"),

    /**
     * Document specifying an accounting statement.
     */
    V_832("832", "Accounting statement"),

    /**
     * Generic term, sometimes referred to as Freight declaration, applied to the documents
     * providing the particulars required by the Customs concerning the cargo (freight) carried
     * by commercial means of transport (CCC).
     */
    V_833("833", "Cargo declaration (departure)"),

    /**
     * Document confirming the receipt of a payment.
     */
    V_834("834", "Payment receipt confirmation"),

    /**
     * Document certifying the payment of the insurance premium.
     */
    V_835("835", "Certificate of paid insurance premium"),

    /**
     * Report about payments done towards an insured party.
     */
    V_836("836", "Insured party payment report"),

    /**
     * Report about payments done towards a third party.
     */
    V_837("837", "Third party payment report"),

    /**
     * Document giving the addressee the right to debit from an account of the authorizing
     * party.
     */
    V_838("838", "Direct debit authorisation"),

    /**
     * Report issued by a medical doctor.
     */
    V_839("839", "Physician report"),

    /**
     * Document/message submitted to a competent body by party requesting a Goods control certificate
     * to be issued in accordance with national or international standards, or conforming to
     * legislation in the importing country, or as specified in the contract.
     */
    V_840("840", "Application for goods control certificate"),

    /**
     * Document/message issued by a competent body evidencing the quality of the goods described
     * therein, in accordance with national or international standards, or conforming to legislation
     * in the importing country, or as specified in the contract.
     */
    V_841("841", "Goods control certificate"),

    /**
     * Document certifying a medical condition.
     */
    V_842("842", "Medical certificate"),

    /**
     * Document containing a report of a witness.
     */
    V_843("843", "Witness report"),

    /**
     * Document detailing a calculation, such as an invoice calculation or a costs calculation.
     */
    V_844("844", "Calculation note"),

    /**
     * Document containing a communication from the opposite party, such as in legal action.
     */
    V_845("845", "Communication from opposite party"),

    /**
     * Document specifying an amicable agreement.
     */
    V_846("846", "Amicable agreement"),

    /**
     * Document which specifies an out of court settlement.
     */
    V_847("847", "Out of court settlement"),

    /**
     * Document specifying a legal action at court.
     */
    V_848("848", "Legal action"),

    /**
     * Document specifying a summons to court.
     */
    V_849("849", "Summons"),

    /**
     * Document/message submitted to a competent body by party requesting a Phytosanitary certificate
     * to be issued.
     */
    V_850("850", "Application for phytosanitary certificate"),

    /**
     * A message/doucment consistent with the model for certificates of the IPPC, attesting
     * that a consignment meets phytosanitary import requirements.
     */
    V_851("851", "Phytosanitary certificate"),

    /**
     * Document/message issued by the competent authority in the exporting country evidencing
     * that alimentary and animal products, including dead animals, are fit for human consumption,
     * and giving details, when relevant, of controls undertaken.
     */
    V_852("852", "Sanitary certificate"),

    /**
     * Document/message issued by the competent authority in the exporting country evidencing
     * that live animals or birds are not infested or infected with disease, and giving details
     * regarding their provenance, and of vaccinations and other treatment to which they have
     * been subjected.
     */
    V_853("853", "Veterinary certificate"),

    /**
     * Document specifying a judgment of a court.
     */
    V_854("854", "Court judgment"),

    /**
     * Document/message submitted to a competent body by a party requesting an Inspection certificate
     * to be issued in accordance with national or international standards, or conforming to
     * legislation in the country in which it is required, or as specified in the contract.
     */
    V_855("855", "Application for inspection certificate"),

    /**
     * Document/message issued by a competent body evidencing that the goods described therein
     * have been inspected in accordance with national or international standards, in conformity
     * with legislation in the country in which the inspection is required, or as specified
     * in the contract.
     */
    V_856("856", "Inspection certificate"),

    /**
     * Document which must be aboard the vehicle.
     */
    V_857("857", "Vehicle aboard document"),

    /**
     * Document consisting of an image.
     */
    V_858("858", "Image"),

    /**
     * Document consisting of an audio recording (e.g. a telephone conversation or alike).
     */
    V_859("859", "Audio"),

    /**
     * Document/message submitted to a competent body by an interested party requesting a Certificate
     * of origin to be issued in accordance with relevant criteria, and on the basis of evidence
     * of the origin of the goods.
     */
    V_860("860", "Certificate of origin, application for"),

    /**
     * Document/message identifying goods, in which the authority or body authorized to issue
     * it certifies expressly that the goods to which the certificate relates originate in
     * a specific country. The word "country" may include a group of countries, a region or
     * a part of a country. This certificate may also include a declaration by the manufacturer,
     * producer, supplier, exporter or other competent person.
     */
    V_861("861", "Certificate of origin"),

    /**
     * Appropriate statement as to the origin of the goods, made in connection with their exportation
     * by the manufacturer, producer, supplier, exporter or other competent person on the Commercial
     * invoice or any other document relating to the goods (CCC).
     */
    V_862("862", "Declaration of origin"),

    /**
     * Certificate drawn up in accordance with the rules laid down by an authority or approved
     * body, certifying that the goods described therein qualify for a designation specific
     * to the given region (e.g. champagne, port wine, Parmesan cheese).
     */
    V_863("863", "Regional appellation certificate"),

    /**
     * Document/message describing a certificate of origin meeting the requirements for preferential
     * treatment.
     */
    V_864("864", "Preference certificate of origin"),

    /**
     * Specific form of certificate of origin for goods qualifying for preferential treatment
     * under the generalized system of preferences (includes a combined declaration of origin
     * and certificate, form A).
     */
    V_865("865", "Certificate of origin form GSP"),

    /**
     * Document consisting of a video.
     */
    V_866("866", "Video"),

    /**
     * A letter of introduction attached to, or accompanying another document such as an insurance
     * policy.
     */
    V_867("867", "Introductory letter"),

    /**
     * Document specifying the terms of data protection regulations.
     */
    V_868("868", "Data protection regulations statement"),

    /**
     * Document expressing the mandate of a client for a service only by the mandated broker.
     */
    V_869("869", "Exclusive brokerage mandate"),

    /**
     * Document/message to be prepared by an exporter in his country and presented to a diplomatic
     * representation of the importing country for endorsement and subsequently to be presented
     * by the importer in connection with the import of the goods described therein.
     */
    V_870("870", "Consular invoice"),

    /**
     * Document expressing the mandate of a client for an inquiry service by the mandated provider.
     */
    V_871("871", "Inquiry mandate"),

    /**
     * Document specifying the analysis of risks.
     */
    V_872("872", "Risk analysis"),

    /**
     * A partial transport equipment movement report, containing only a selected part of the
     * movements of transport equipment for a vessel in a port.
     */
    V_873("873", "Transport equipment movement report, partial"),

    /**
     * Declaration of the conveyance to a public authority.
     */
    V_874("874", "Conveyance declaration"),

    /**
     * Partial invoice in the context of a specific construction project.
     */
    V_875("875", "Partial construction invoice"),

    /**
     * Invoice concluding all previous partial construction invoices of a completed partial
     * rendered service in the context of a specific construction project.
     */
    V_876("876", "Partial final construction invoice"),

    /**
     * Invoice concluding all previous partial invoices and partial final construction invoices
     * in the context of a specific construction project.
     */
    V_877("877", "Final construction invoice"),

    /**
     * Certificate issued to business that fulfils specified criteria applied to the security
     * and safety of the logistics chain in the flow of foreign trade operations by a national
     * AEO recognized program (e.g. AEO-Security and Safety (AEOS) - Regulation (EU) No 952/2013).
     */
    V_878("878", "AEO Certificate of Security and/or Safety"),

    /**
     * Certificate issued to business that fulfils specified criteria for compliance with tax
     * and customs obligations, as well as financial solvency by a national AEO recognized
     * program (e.g. AEO-Customs Simplifications (AEOC) - Regulation (EU) No 952/2013).
     */
    V_879("879", "AEO Certificate of Conformity or Compliance"),

    /**
     * (1115) Document/message issued by a consignor in accordance with applicable conventions
     * or regulations, describing hazardous goods or materials for transport purposes, and
     * stating that the latter have been packed and labelled in accordance with the provisions
     * of the relevant conventions or regulations.
     */
    V_890("890", "Dangerous goods declaration"),

    /**
     * Certificate issued to business that fulfils specified criteria to both AEO Certificate
     * of Security and/or Safety and AEO Certificate of Conformity or Compliance by a national
     * AEO recognized program (e.g. AEO-Customs Simplifications/Security and Safety (AEOC/AEOS)
     * - Regulation(EU) No 952/2013).
     */
    V_891("891", "AEO Certificate Full"),

    /**
     * Document enabling the Financing Requestor to initiate the financing process by the First
     * Agent.
     */
    V_892("892", "Purchase Order Financing Request"),

    /**
     * Document enabling the First Agent to notify the Financing Requestor of the status of
     * a purchase order financing request or the status of a purchase order financing cancellation
     * request previously sent by the Financial Requestor itself.
     */
    V_893("893", "Purchase Order Financing Request Status"),

    /**
     * Document enabling the Financing Requestor to request the First Agent to cancel a previously
     * sent purchase order financing request.
     */
    V_894("894", "Purchase Order Financing Request Cancellation"),

    /**
     * Document/message in which an exporter provides information about exported goods required
     * by the body responsible for the collection of international trade statistics.
     */
    V_895("895", "Statistical document, export"),

    /**
     * Document/message in which a declarant provides information about goods required by the
     * body responsible for the collection of trade statistics.
     */
    V_896("896", "INTRASTAT declaration"),

    /**
     * Certificate of approval for vehicles and containers used to transit goods under customs
     * seals.
     */
    V_897("897", "Transit certificate of approval"),

    /**
     * A consignment order requesting the supply of products packed according to the final
     * delivery point which will be moved across a dock in a distribution centre without further
     * handling.
     */
    V_898("898", "Pre-packed cross docking consignment order"),

    /**
     * Document/message declaring a traceability event.
     */
    V_899("899", "Traceability event declaration"),

    /**
     * Document/message requesting information based on defined criteria regarding sustainability.
     */
    V_900("900", "Sustainability data request"),

    /**
     * Document/message whereby an official authority (Customs or governmental) certifies that
     * goods have been delivered.
     */
    V_901("901", "Delivery verification certificate"),

    /**
     * Document/Message returned as an answer to a question regarding sustainability.
     */
    V_902("902", "Sustainability data response"),

    /**
     * Document/message requesting a sustainability inspection.
     */
    V_903("903", "Sustainability Inspection request"),

    /**
     * Document/message reporting the results of a sustainability inspection.
     */
    V_904("904", "Sustainability Inspection response"),

    /**
     * Message to provide the forecast information about ships, trains, vehicles and aircrafts
     * arrival at the destination.
     */
    V_905("905", "Transport Means Forecast Information Message"),

    /**
     * Message to provide the actual information about ships, trains, vehicles and aircrafts
     * arrival at the destination.
     */
    V_906("906", "Transport Means Actual Information Message"),

    /**
     * Message to provide the arrival information of transport means and goods.
     */
    V_907("907", "Arrival Report Message"),

    /**
     * Message to provide the information of goods tallying.
     */
    V_908("908", "Tally Message"),

    /**
     * Message to provide the information of goods loading.
     */
    V_909("909", "Goods Loading Message"),

    /**
     * Document/message in which an interested party applies to the competent body for authorization
     * to import either a limited quantity of articles subject to import restrictions, or an
     * unlimited quantity of such articles during a limited period, and specifies the kind
     * of articles, their origin and value, etc.
     */
    V_910("910", "Import licence, application for"),

    /**
     * Document/message issued by the competent body in accordance with import regulations
     * in force, by which authorization is granted to a named party to import either a limited
     * quantity of designated articles or an unlimited quantity of such articles during a limited
     * period, under conditions specified in the document.
     */
    V_911("911", "Import licence"),

    /**
     * Message to provide the releasing information of transport means and goods.
     */
    V_912("912", "Transport Means and Goods Release Message."),

    /**
     * CUSDEC transmission that does not include data from the commercial detail section of
     * the message.
     */
    V_913("913", "Customs declaration without commercial detail"),

    /**
     * CUSDEC transmission that includes data from both the commercial detail and item detail
     * sections of the message.
     */
    V_914("914", "Customs declaration with commercial and item detail"),

    /**
     * CUSDEC transmission that does not include data from the item detail section of the message.
     */
    V_915("915", "Customs declaration without item detail"),

    /**
     * Document that has a relationship with the stated document/message.
     */
    V_916("916", "Related document"),

    /**
     * Receipt for Customs duty/tax/fee paid.
     */
    V_917("917", "Receipt (Customs)"),

    /**
     * Document/message whereby an importer/buyer requests the competent body to allocate an
     * amount of foreign exchange to be transferred to an exporter/seller in payment for goods.
     */
    V_925("925", "Application for exchange allocation"),

    /**
     * Document/message issued by the competent body authorizing an importer/buyer to transfer
     * an amount of foreign exchange to an exporter/seller in payment for goods.
     */
    V_926("926", "Foreign exchange permit"),

    /**
     * Document/message completed by an importer/buyer as a means for the competent body to
     * control that a trade transaction for which foreign exchange has been allocated has been
     * executed and that money has been transferred in accordance with the conditions of payment
     * and the exchange control regulations in force.
     */
    V_927("927", "Exchange control declaration (import)"),

    /**
     * Document/message by which goods are declared for import Customs clearance [sister entry
     * of 830].
     */
    V_929("929", "Goods declaration for importation"),

    /**
     * Document/message by which goods are declared for import Customs clearance according
     * to Annex B.1 (concerning clearance for home use) to the Kyoto convention (CCC).
     */
    V_930("930", "Goods declaration for home use"),

    /**
     * Document/message issued by an importer notifying Customs that goods have been removed
     * from an importing means of transport to the importer's premises under a Customs-approved
     * arrangement for immediate release, or requesting authorization to do so.
     */
    V_931("931", "Customs immediate release declaration"),

    /**
     * Document/message whereby a Customs authority releases goods under its control to be
     * placed at the disposal of the party concerned. Synonym: Customs release note.
     */
    V_932("932", "Customs delivery note"),

    /**
     * Generic term, sometimes referred to as Freight declaration, applied to the documents
     * providing the particulars required by the Customs concerning the cargo (freight) carried
     * by commercial means of transport (CCC).
     */
    V_933("933", "Cargo declaration (arrival)"),

    /**
     * Document/message in which a declarant (importer) states the invoice or other price (e.g.
     * selling price, price of identical goods), and specifies costs for freight, insurance
     * and packing, etc., terms of delivery and payment, any relationship with the trading
     * partner, etc., for the purpose of determining the Customs value of goods imported.
     */
    V_934("934", "Value declaration"),

    /**
     * Document/message required by the Customs in an importing country in which an exporter
     * states the invoice or other price (e.g. selling price, price of identical goods), and
     * specifies costs for freight, insurance and packing, etc., terms of delivery and payment,
     * for the purpose of determining the Customs value in the importing country of goods consigned
     * to that country.
     */
    V_935("935", "Customs invoice"),

    /**
     * Document/message which, according to Article 106 of the "Agreement concerning Postal
     * Parcels" under the UPU Convention, must accompany post parcels and in which the contents
     * of such parcels are specified.
     */
    V_936("936", "Customs declaration (post parcels)"),

    /**
     * Document/message in which an importer states the pertinent information required by the
     * competent body for assessment of value-added tax.
     */
    V_937("937", "Tax declaration (value added tax)"),

    /**
     * Document/message containing a general tax declaration.
     */
    V_938("938", "Tax declaration (general)"),

    /**
     * Document/message containing the demand of tax.
     */
    V_940("940", "Tax demand"),

    /**
     * Document/message giving the permission to export specified goods.
     */
    V_941("941", "Embargo permit"),

    /**
     * Document/message by which the sender declares goods for Customs transit according to
     * Annex E.1 (concerning Customs transit) to the Kyoto convention (CCC).
     */
    V_950("950", "Goods declaration for Customs transit"),

    /**
     * International Customs transit document by which the sender declares goods for carriage
     * by rail in accordance with the provisions of the 1952 International Convention to facilitate
     * the crossing of frontiers for goods carried by rail (TIF Convention of UIC).
     */
    V_951("951", "TIF form"),

    /**
     * International Customs document (International Transit by Road), issued by a guaranteeing
     * association approved by the Customs authorities, under the cover of which goods are
     * carried, in most cases under Customs seal, in road vehicles and/or containers in compliance
     * with the requirements of the Customs TIR Convention of the International Transport of
     * Goods under cover of TIR Carnets (UN/ECE).
     */
    V_952("952", "TIR carnet"),

    /**
     * EC customs transit document issued by EC customs authorities for transit and/or temporary
     * user of goods within the EC.
     */
    V_953("953", "EC carnet"),

    /**
     * Customs certificate used in preferential goods interchanges between EC countries and
     * EC external countries.
     */
    V_954("954", "EUR 1 certificate of origin"),

    /**
     * International Customs document (Admission Temporaire / Temporary Admission) which, issued
     * under the terms of the ATA Convention (1961), incorporates an internationally valid
     * guarantee and may be used, in lieu of national Customs documents and as security for
     * import duties and taxes, to cover the temporary admission of goods and, where appropriate,
     * the transit of goods. If accepted for controlling the temporary export and reimport
     * of goods, international guarantee does not apply (CCC).
     */
    V_955("955", "ATA carnet"),

    /**
     * A set of documents, replacing the various (national) forms for Customs declaration within
     * the EC, implemented on 01-01-1988.
     */
    V_960("960", "Single administrative document"),

    /**
     * General response message to permit the transfer of data from Customs to the transmitter
     * of the previous message.
     */
    V_961("961", "General response (Customs)"),

    /**
     * Document response message to permit the transfer of data from Customs to the transmitter
     * of the previous message.
     */
    V_962("962", "Document response (Customs)"),

    /**
     * Error response message to permit the transfer of data from Customs to the transmitter
     * of the previous message.
     */
    V_963("963", "Error response (Customs)"),

    /**
     * Package response message to permit the transfer of data from Customs to the transmitter
     * of the previous message.
     */
    V_964("964", "Package response (Customs)"),

    /**
     * Tax calculation/confirmation response message to permit the transfer of data from Customs
     * to the transmitter of the previous message.
     */
    V_965("965", "Tax calculation/confirmation response (Customs)"),

    /**
     * Document/message issued by the competent body for prior allocation of a quota.
     */
    V_966("966", "Quota prior allocation certificate"),

    /**
     * Document which contains consignment information concerning the wagons and their lading
     * in a case of a multiple wagon consignment.
     */
    V_970("970", "Wagon report"),

    /**
     * Document for a course of transit used for a carrier who is neither the carrier at the
     * beginning nor the arrival. The transit carrier can directly invoice the expenses for
     * its part of the transport.
     */
    V_971("971", "Transit Conveyor Document"),

    /**
     * Document which is a copy of the rail consignment note printed especially for the need
     * of the forwarder.
     */
    V_972("972", "Rail consignment note forwarder copy"),

    /**
     * Document giving details for the carriage of excisable goods on a duty-suspended basis.
     */
    V_974("974", "Duty suspended goods"),

    /**
     * A document providing proof that a transit declaration has been accepted.
     */
    V_975("975", "Proof of transit declaration"),

    /**
     * Document for the carriage of containers. Syn: transfer note.
     */
    V_976("976", "Container transfer note"),

    /**
     * Customs transit document for the carriage of shipments of the NATO armed forces under
     * Customs supervision.
     */
    V_977("977", "NATO transit document"),

    /**
     * Document containing the authorization from the relevant authority for the international
     * carriage of waste. Syn: Transfrontier waste shipment permit.
     */
    V_978("978", "Transfrontier waste shipment authorization"),

    /**
     * Document certified by the carriers and the consignee to be used for the international
     * carriage of waste.
     */
    V_979("979", "Transfrontier waste shipment movement document"),

    /**
     * Document issued by Customs granting the end-use Customs procedure.
     */
    V_990("990", "End use authorization"),

    /**
     * Document/message describing a contract with a government authority.
     */
    V_991("991", "Government contract"),

    /**
     * Document/message describing an import document that is used for statistical purposes.
     */
    V_995("995", "Statistical document, import"),

    /**
     * Message with application for opening of a documentary credit.
     */
    V_996("996", "Application for documentary credit"),

    /**
     * Indication of the previous Customs document/message concerning the same transaction.
     */
    V_998("998", "Previous Customs document/message"),
    ;

    private final String name;
    private final String code;

    DocumentCodeType(String code, String name) {
        this.code = code;
        this.name = name;
    }

    @Override
    public String getName() {
        return name;
    }

    @Override
    public String getCode() {
        return code;
    }

}
